
<p><div style="wigth:350px"><a class="internal" href="/wiki/Image:350px-Rna_interference_plos.png" title="The enzyme [[dicer]] trims double stranded RNA, to form [[small interfering RNA]] or [[microRNA]]. These processed RNAs are incorporated into the [[RNA-induced silencing complex]] (RISC), which targets [[messenger RNA]] to prevent [[translation (genetics)|translation]].&#60;ref name=&#34;Hammond2000&#34;&#62;{{cite journal}}&#60;/ref&#62;"><img src="http://en.wikipedia.org/wiki/350px-Rna_interference_plos.png" alt="The enzyme [[dicer]] trims double stranded RNA, to form [[small interfering RNA]] or [[microRNA]]. These processed RNAs are incorporated into the [[RNA-induced silencing complex]] (RISC), which targets [[messenger RNA]] to prevent [[translation (genetics)|translation]].&#60;ref name=&#34;Hammond2000&#34;&#62;{{cite journal}}&#60;/ref&#62;" title="The enzyme [[dicer]] trims double stranded RNA, to form [[small interfering RNA]] or [[microRNA]]. These processed RNAs are incorporated into the [[RNA-induced silencing complex]] (RISC), which targets [[messenger RNA]] to prevent [[translation (genetics)|translation]].&#60;ref name=&#34;Hammond2000&#34;&#62;{{cite journal}}&#60;/ref&#62;" class="location-right type-thumb" width="350px" />
</a>
<div class="thumbcaption">The enzyme <a href="/wiki/Dicer" id="w">dicer</a> trims double stranded RNA, to form <a href="/wiki/Small_interfering_RNA" id="w">small interfering RNA</a> or <a href="/wiki/MicroRNA" id="w">microRNA</a>. These processed RNAs are incorporated into the <a href="/wiki/RNA-induced_silencing_complex" id="w">RNA-induced silencing complex</a> (RISC), which targets <a href="/wiki/Messenger_RNA" id="w">messenger RNA</a> to prevent <a href="/wiki/Translation_(genetics)" id="w">translation</a>.<sup id="_ref-Hammond2000_a" class="reference"><a href="#_note-Hammond2000" title="">[1]</a></sup></div></div>
 </p>
<p><b>RNA interference</b> (<b>RNAi</b>) is a mechanism that inhibits <a href="/wiki/Gene_expression" id="w">gene expression</a> by causing the degradation of specific <a href="/wiki/RNA" id="w">RNA</a> molecules or hindering the <a href="/wiki/Transcription_(genetics)" id="w">transcription</a> of specific genes. The targets are often RNA from <a href="/wiki/Virus" id="w">viruses</a> and <a href="/wiki/Transposon" id="w">transposons</a> (probably a form of innate <a href="/wiki/Immune__system" id="w">immune response</a>), although it also plays a role in <a href="/wiki/Gene_regulation" id="w">regulating</a> <a href="/wiki/Developmental_biology" id="w">development</a> and <a href="/wiki/Genome" id="w">genome</a> maintenance. Key to the RNAi processes are <a href="/wiki/Small_interfering_RNA" id="w">small interfering RNA</a> strands (siRNA), which have <a href="/wiki/Complementarity_(molecular_biology)" id="w">complementary</a> <a href="/wiki/Nucleotide" id="w">nucleotide</a> sequences to a targeted RNA strand. The siRNA &#34;guides&#34; proteins within the RNAi pathway to the targeted <a href="/wiki/Messenger_RNA" id="w">messenger RNA</a> (mRNA) and &#34;cleaves&#34; them, breaking them down into smaller portions that can no longer be translated into protein. A type of RNA transcribed from the genome itself, <a href="/wiki/MicroRNA" id="w">miRNA</a>, works in the same way.</p>
<p>The RNAi pathway is initiated by the enzyme <a href="/wiki/Dicer" id="w">dicer</a>, which cleaves long, double-stranded RNA (dsRNA) molecules into short fragments of 20&#8211;25 <a href="/wiki/Base_pair" id="w">base pairs</a>. One of the two strands of each fragment, known as the <i>guide strand</i>, is then incorporated into the <a href="/wiki/RNA-induced_silencing_complex" id="w">RNA-induced silencing complex</a> (RISC) and pairs with complementary sequences. The most well-studied outcome of this recognition event is post-transcriptional <a href="/wiki/Gene_silencing" id="w">gene silencing</a>. This occurs when the guide strand specifically pairs with a mRNA molecule and induces the degradation by <a href="/wiki/Argonaute" id="w">argonaute</a>, the catalytic component of the RISC complex.  Another outcome is <a href="/wiki/Epigenetic" id="w">epigenetic</a> changes to a gene – <a href="/wiki/Histone" id="w">histone</a> modification and <a href="/wiki/DNA_methylation" id="w">DNA methylation</a> –  affecting the degree the gene is transcribed.</p>
<p>The selective and robust effect of RNAi on gene expression makes it a valuable research tool, both in <a href="/wiki/Cell_culture" id="w">cell culture</a> and in living organisms because synthetic dsRNA introduced into cells can induce suppression of specific genes of interest. RNAi may also be used for large-scale screens that systematically shut down each gene in the cell, which can help identify the components necessary for a particular cellular process or an event such as <a href="/wiki/Cell_division" id="w">cell division</a>. Exploitation of the pathway is also a promising tool in <a href="/wiki/Biotechnology" id="w">biotechnology</a> and <a href="/wiki/Medicine" id="w">medicine</a>.</p>
<p>Historically, RNA interference was known by other names, including <a href="/wiki/Post_transcriptional_gene_silencing" id="w">post transcriptional gene silencing</a>, transgene silencing, and quelling. Only after these apparently-unrelated processes were fully understood did it become clear that they all described the RNAi phenomenon. RNAi has also been confused with <a href="/wiki/Antisense" id="w">antisense</a> suppression of gene expression, which does not act catalytically to degrade mRNA, but instead involves single-stranded RNA fragments physically binding to mRNA and blocking protein <a href="/wiki/Translation_(biology)" id="w">translation</a>. In 2006, <a href="/wiki/Andrew_Fire" id="w">Andrew Fire</a> and <a href="/wiki/Craig_C._Mello" id="w">Craig C. Mello</a> shared the <a href="/wiki/Nobel_Prize_in_Physiology_or_Medicine" id="w">Nobel Prize in Physiology or Medicine</a> for their work on RNA interference in the <a href="/wiki/Nematode" id="w">nematode</a> worm <i>C. elegans</i>,<sup id="_ref-Daneholt2006_a" class="reference"><a href="#_note-Daneholt2006" title="">[2]</a></sup> which they published in 1998.<sup id="_ref-Fire_a" class="reference"><a href="#_note-Fire" title="">[3]</a></sup> </p>
<a id="Cellular_mechanism" name="Cellular_mechanism"></a><h1>Cellular mechanism</h1>
<p><a class="internal" href="/wiki/Image:2ffl-by-domain.png" title="The [[dicer]] protein from &#39;&#39;[[Giardia intestinalis]]&#39;&#39;, which catalyzes the cleavage of dsRNA to siRNAs. The [[RNase]] domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue.&#60;ref name=&#34;Macrae&#34;&#62;{{cite journal}}&#60;/ref&#62;"><img src="http://en.wikipedia.org/wiki/2ffl-by-domain.png" alt="The [[dicer]] protein from &#39;&#39;[[Giardia intestinalis]]&#39;&#39;, which catalyzes the cleavage of dsRNA to siRNAs. The [[RNase]] domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue.&#60;ref name=&#34;Macrae&#34;&#62;{{cite journal}}&#60;/ref&#62;" title="The [[dicer]] protein from &#39;&#39;[[Giardia intestinalis]]&#39;&#39;, which catalyzes the cleavage of dsRNA to siRNAs. The [[RNase]] domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue.&#60;ref name=&#34;Macrae&#34;&#62;{{cite journal}}&#60;/ref&#62;" class="location-right type-thumb" />
</a>
<div class="thumbcaption">The <a href="/wiki/Dicer" id="w">dicer</a> protein from <i><a href="/wiki/Giardia_intestinalis" id="w">Giardia intestinalis</a></i>, which catalyzes the cleavage of dsRNA to siRNAs. The <a href="/wiki/RNase" id="w">RNase</a> domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue.<sup id="_ref-Macrae_a" class="reference"><a href="#_note-Macrae" title="">[4]</a></sup></div></p>
<p>RNAi is an RNA-dependent <a href="/wiki/Gene_silencing" id="w">gene silencing</a> process that is controlled by the <a href="/wiki/RNA-induced_silencing_complex" id="w">RNA-induced silencing complex</a> (RISC) and is initiated by short double-stranded RNA molecules in a cell&#39;s <a href="/wiki/Cytoplasm" id="w">cytoplasm</a>, where they interact with the catalytic RISC component <a href="/wiki/Argonaute" id="w">argonaute</a>. When the dsRNA is exogenous (coming from infection by a <a href="/wiki/Virus" id="w">virus</a> with an RNA genome or laboratory manipulations), the RNA is imported directly into the cytoplasm and cleaved to short fragments by the enzyme <a href="/wiki/Dicer" id="w">dicer</a>. The initiating dsRNA can also be endogenous (originating in the cell), as in pre-<a href="/wiki/MicroRNA" id="w">microRNAs</a> expressed from <a href="/wiki/Non-coding_RNA" id="w">RNA-coding genes</a> in the genome. The primary transcripts from such genes are first processed to form the characteristic <a href="/wiki/Stem-loop" id="w">stem-loop</a> structure of pre-miRNA in the <a href="/wiki/Cell_nucleus" id="w">nucleus</a>, then exported to the cytoplasm to be cleaved by dicer. Thus the two pathways for exogenous and endogenous dsRNA converge at the RISC complex, which mediates gene silencing effects.</p>
<a id="dsRNA_cleavage" name="dsRNA_cleavage"></a><h2>dsRNA cleavage</h2>
<p>Exogenous dsRNA initiates RNAi by activating the <a href="/wiki/Ribonuclease" id="w">ribonuclease</a> protein <a href="/wiki/Dicer" id="w">dicer</a>,<sup id="_ref-Bernstein_a" class="reference"><a href="#_note-Bernstein" title="">[5]</a></sup> which binds and cleaves double-stranded RNAs (dsRNA)s to produce double-stranded fragments of 20–25 <a href="/wiki/Base_pair" id="w">base pairs</a> with a few unpaired overhang bases on each end.<sup id="_ref-Zamore_a" class="reference"><a href="#_note-Zamore" title="">[6]</a></sup><sup id="_ref-Vermeulen_a" class="reference"><a href="#_note-Vermeulen" title="">[7]</a></sup> <a href="/wiki/Bioinformatics" id="w">Bioinformatics</a> studies on the genomes of multiple organisms suggest this length maximizes target-gene specificity and minimizes non-specific effects.<sup id="_ref-Qiu_a" class="reference"><a href="#_note-Qiu" title="">[8]</a></sup> These short double-stranded fragments are called small interfering RNAs (<a href="/wiki/SiRNA" id="w">siRNAs</a>). These siRNAs are then separated into single strands and integrated into an active RISC complex. After integration into the RISC, siRNAs base-pair to their target mRNA and induce cleavage of the mRNA, thereby preventing it from being used as a <a href="/wiki/Translation_(genetics)" id="w">translation</a> template. </p>
<p>Exogenous dsRNA is detected and bound by an effector protein, known as RDE-4 in <i><a href="/wiki/C._elegans" id="w">C. elegans</a></i> and R2D2 in <i><a href="/wiki/Drosophila" id="w">Drosophila</a></i>, that stimulates dicer activity.<sup id="_ref-Parker_a" class="reference"><a href="#_note-Parker" title="">[9]</a></sup> This protein only binds long dsRNAs, but the mechanism producing this length specificity is unknown.<sup id="_ref-Parker_b" class="reference"><a href="#_note-Parker" title="">[9]</a></sup> These RNA-binding proteins then facilitate transfer of cleaved siRNAs to the RISC complex.<sup id="_ref-Liu2003_a" class="reference"><a href="#_note-Liu2003" title="">[10]</a></sup> </p>
<p>This initiation pathway may be amplified by the cell through the synthesis of a population of &#39;secondary&#39; siRNAs using the dicer-produced initiating or &#39;primary&#39; siRNAs as templates.<sup id="_ref-11" class="reference"><a href="#_note-11" title="">[11]</a></sup> These siRNAs are structurally distinct from dicer-produced siRNAs and appear to be produced by an <a href="/wiki/RNA-dependent_RNA_polymerase" id="w">RNA-dependent RNA polymerase</a> (RdRP).<sup id="_ref-Pak_a" class="reference"><a href="#_note-Pak" title="">[12]</a></sup><sup id="_ref-Sijen_a" class="reference"><a href="#_note-Sijen" title="">[13]</a></sup></p>
<a id="microRNA" name="microRNA"></a><h2>microRNA</h2>
<p><a href="/wiki/MicroRNA" id="w">MicroRNAs</a> (miRNAs) are <a href="/wiki/Genome" id="w">genomically</a> encoded <a href="/wiki/Non-coding_RNA" id="w">non-coding RNAs</a> that help regulate <a href="/wiki/Gene_expression" id="w">gene expression</a>, particularly during <a href="/wiki/Developmental_biology" id="w">development</a>. The phenomenon of RNA interference, broadly defined, includes the endogenously induced gene silencing effects of miRNAs as well as silencing triggered by foreign dsRNA. Mature miRNAs are structurally similar to siRNAs produced from exogenous dsRNA, but before reaching maturity, miRNAs must first undergo extensive <a href="/wiki/Post-transcriptional_modification" id="w">post-transcriptional modification</a>. An miRNA is expressed from a much longer RNA-coding gene as a <a href="/wiki/Primary_transcript" id="w">primary transcript</a> known as a <i>pri-miRNA</i>, which is processed in the <a href="/wiki/Cell_nucleus" id="w">cell nucleus</a> to a 70-nucleotide <a href="/wiki/Stem-loop" id="w">stem-loop</a> structure called a <i>pre-miRNA</i> by the microprocessor complex. This complex consists of an <a href="/wiki/RNase_III" id="w">RNase III</a> enzyme called <a href="/wiki/Drosha" id="w">Drosha</a> and a dsRNA-binding protein <a href="/wiki/Pasha" id="w">Pasha</a>. The dsRNA portion of this pre-miRNA is bound and cleaved by dicer to produce the mature miRNA molecule that can be integrated into the RISC complex; thus, miRNA and siRNA share the same cellular machinery downstream of their initial processing.<sup id="_ref-Denli_a" class="reference"><a href="#_note-Denli" title="">[14]</a></sup> </p>
<p>The siRNAs derived from long dsRNA precursors differ from miRNAs in that miRNAs, especially those in animals, typically have incomplete base pairing to a target and inhibit the translation of many different mRNAs with similar sequences. In contrast, siRNAs typically base-pair perfectly and induce mRNA cleavage only in a single, specific target.<sup id="_ref-Pillai_2006_a" class="reference"><a href="#_note-Pillai_2006" title="">[15]</a></sup> In <i><a href="/wiki/Drosophila" id="w">Drosophila</a></i> and <i><a href="/wiki/C._elegans" id="w">C. elegans</a></i>, miRNA and siRNA are processed by distinct argonaute proteins and dicer enzymes.<sup id="_ref-Okamura_a" class="reference"><a href="#_note-Okamura" title="">[16]</a></sup><sup id="_ref-Lee_a" class="reference"><a href="#_note-Lee" title="">[17]</a></sup></p>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Argonaute_1u04_1ytu_composite.png" title="&#39;&#39;Left:&#39;&#39; A full-length [[argonaute]] protein from the [[archaea]] species &#39;&#39;[[Pyrococcus furiosus]]&#39;&#39;. &#39;&#39;Right:&#39;&#39; The [[PIWI domain]] of an [[argonaute]] protein in complex with [[double-stranded RNA]]."><img src="http://en.wikipedia.org/wiki/300px-Argonaute_1u04_1ytu_composite.png" alt="&#39;&#39;Left:&#39;&#39; A full-length [[argonaute]] protein from the [[archaea]] species &#39;&#39;[[Pyrococcus furiosus]]&#39;&#39;. &#39;&#39;Right:&#39;&#39; The [[PIWI domain]] of an [[argonaute]] protein in complex with [[double-stranded RNA]]." title="&#39;&#39;Left:&#39;&#39; A full-length [[argonaute]] protein from the [[archaea]] species &#39;&#39;[[Pyrococcus furiosus]]&#39;&#39;. &#39;&#39;Right:&#39;&#39; The [[PIWI domain]] of an [[argonaute]] protein in complex with [[double-stranded RNA]]." class="location-left type-thumb" width="300px" />
</a>
<div class="thumbcaption"><i>Left:</i> A full-length <a href="/wiki/Argonaute" id="w">argonaute</a> protein from the <a href="/wiki/Archaea" id="w">archaea</a> species <i><a href="/wiki/Pyrococcus_furiosus" id="w">Pyrococcus furiosus</a></i>. <i>Right:</i> The <a href="/wiki/PIWI_domain" id="w">PIWI domain</a> of an <a href="/wiki/Argonaute" id="w">argonaute</a> protein in complex with <a href="/wiki/Double-stranded_RNA" id="w">double-stranded RNA</a>.</div></div>
</p>
<a id="RISC_activation_and_catalysis" name="RISC_activation_and_catalysis"></a><h2>RISC activation and catalysis</h2>
<p>The active components of an <a href="/wiki/RNA-induced_silencing_complex" id="w">RNA-induced silencing complex</a> (RISC) are <a href="/wiki/Endonuclease" id="w">endonucleases</a> called <a href="/wiki/Argonaute" id="w">argonaute</a> proteins, which cleave the target mRNA strand <a href="/wiki/Complementarity_(molecular_biology)" id="w">complementary</a> to their bound siRNA. As the fragments produced by dicer are double-stranded, they could each in theory produce a functional siRNA. However, only one of the two strands, which is known as the <i>guide strand</i>, binds the argonaute protein and directs gene silencing. The other <i>anti-guide strand</i> or <i>passenger strand</i> is degraded during RISC activation.<sup id="_ref-Gregory_a" class="reference"><a href="#_note-Gregory" title="">[18]</a></sup> Although it was first believed that an <a href="/wiki/Adenosine_triphosphate" id="w">ATP</a>-dependent <a href="/wiki/Helicase" id="w">helicase</a> separated these two strands,<sup id="_ref-Lodish_a" class="reference"><a href="#_note-Lodish" title="">[19]</a></sup> the process is actually ATP-independent and performed directly by the protein components of RISC.<sup id="_ref-Matranga_a" class="reference"><a href="#_note-Matranga" title="">[20]</a></sup><sup id="_ref-Leuschner_a" class="reference"><a href="#_note-Leuschner" title="">[21]</a></sup> The strand selected as the guide tends to be that with a more stable <a href="/wiki/5%26%2339%3B_end" id="w">5&#39; end</a>, but strand selection is unaffected by the direction in which dicer cleaves the dsRNA before RISC incorporation.<sup id="_ref-Preall_a" class="reference"><a href="#_note-Preall" title="">[22]</a></sup> Instead, the R2D2 protein may serve as the differentiating factor by binding the less-stable 5&#39; end of the passenger strand.<sup id="_ref-Tomari_a" class="reference"><a href="#_note-Tomari" title="">[23]</a></sup> </p>
<p>The structural basis for binding of RNA to the argonaute protein was examined by <a href="/wiki/X-ray_crystallography" id="w">X-ray crystallography</a> of the binding <a href="/wiki/Structural_domain" id="w">domain</a> of an RNA-bound argonaute protein. Here, the <a href="/wiki/Phosphorylation" id="w">phosphorylated</a> 5&#39; end of the RNA strand enters a <a href="/wiki/Conservation_(genetics)" id="w">conserved</a> <a href="/wiki/Basicity" id="w">basic</a> surface <a href="/wiki/Binding_site" id="w">pocket</a> and makes contacts through a <a href="/wiki/Divalent" id="w">divalent</a> <a href="/wiki/Cation" id="w">cation</a> (an atom with two or more extra electrons) such as <a href="/wiki/Magnesium" id="w">magnesium</a> and by <a href="/wiki/Aromaticity" id="w">aromatic</a> <a href="/wiki/Stacking_(chemistry)" id="w">stacking</a> (a process that allows more than one atom to share an electron by passing it back and forth) between the 5&#39; nucleotide in the siRNA and a conserved <a href="/wiki/Tyrosine" id="w">tyrosine</a> residue. This site is thought to form a nucleation site for the binding of the siRNA to its mRNA target.<sup id="_ref-Ma_a" class="reference"><a href="#_note-Ma" title="">[24]</a></sup> </p>
<p>It is not understood how the activated RISC complex locates complementary mRNAs within the cell. Although the cleavage process has been proposed to be linked to <a href="/wiki/Translation_(genetics)" id="w">translation</a>, translation of the mRNA target is not essential for RNAi-mediated degradation.<sup id="_ref-Sen_a" class="reference"><a href="#_note-Sen" title="">[25]</a></sup> Indeed, RNAi may be more effective against mRNA targets that are not translated.<sup id="_ref-Gu_a" class="reference"><a href="#_note-Gu" title="">[26]</a></sup> Argonaute proteins, the catalytic components of RISC, are localized to specific regions in the <a href="/wiki/Cytoplasm" id="w">cytoplasm</a> called <a href="/wiki/P-bodies" id="w">P-bodies</a> (also cytoplasmic bodies or GW bodies), which are regions with high rates of mRNA decay;<sup id="_ref-SenBlau_a" class="reference"><a href="#_note-SenBlau" title="">[27]</a></sup> miRNA activity is also clustered in P-bodies.<sup id="_ref-Lian_a" class="reference"><a href="#_note-Lian" title="">[28]</a></sup> The disruption of P bodies in cells decreases the efficiency of RNA interference, suggesting that they are the site of a critical step in the RNAi process.<sup id="_ref-Jakymiw_a" class="reference"><a href="#_note-Jakymiw" title="">[29]</a></sup></p>
<a id="Transcriptional_silencing" name="Transcriptional_silencing"></a><h2>Transcriptional silencing</h2>
<p>Components of the RNA interference pathway are also used in many eukaryotes in the maintenance of the organisation and structure of their <a href="/wiki/Genome" id="w">genomes</a>. Modification of <a href="/wiki/Histone" id="w">histones</a> and associated induction of <a href="/wiki/Heterochromatin" id="w">heterochromatin</a> formation serves to downregulate genes pre-<a href="/wiki/Transcription_(genetics)" id="w">transcriptionally</a>;<sup id="_ref-Holmquist_2006_a" class="reference"><a href="#_note-Holmquist_2006" title="">[30]</a></sup> this process is referred to as <a href="/wiki/RNA-induced_transcriptional_silencing" id="w">RNA-induced transcriptional silencing</a> (RITS), and is carried out by a complex of proteins called the RITS complex. In <a href="/wiki/Fission_yeast" id="w">fission yeast</a> this complex contains <a href="/wiki/Argonaute" id="w">argonaute</a>, a <a href="/wiki/Chromodomain" id="w">chromodomain</a> protein Chp1, and a protein called Tas3 of unknown function.<sup id="_ref-Verdel_a" class="reference"><a href="#_note-Verdel" title="">[31]</a></sup> As a consequence, the induction and spread of heterochromatic regions requires the argonaute and RdRP proteins.<sup id="_ref-Irvine_a" class="reference"><a href="#_note-Irvine" title="">[32]</a></sup> Indeed, deletion of these genes in the fission yeast <i><a href="/wiki/Schizosaccharomyces_pombe" id="w">S. pombe</a></i> disrupts <a href="/wiki/Histone_methylation" id="w">histone methylation</a> and <a href="/wiki/Centromere" id="w">centromere</a> formation,<sup id="_ref-Volpe_2002_a" class="reference"><a href="#_note-Volpe_2002" title="">[33]</a></sup> causing slow or stalled <a href="/wiki/Anaphase" id="w">anaphase</a> during <a href="/wiki/Cell_division" id="w">cell division</a>.<sup id="_ref-Volpe_2003_a" class="reference"><a href="#_note-Volpe_2003" title="">[34]</a></sup> In some cases, similar processes associated with histone modification have been observed to transcriptionally upregulate genes.<sup id="_ref-Li_a" class="reference"><a href="#_note-Li" title="">[35]</a></sup></p>
<p>The mechanism by which the RITS complex induces heterochromatin formation and organization is not well understood, and most studies have focused on the <a href="/wiki/Mating-type_region" id="w">mating-type region</a> in fission yeast, which may not be representative of activities in other genomic regions or organisms. In maintenance of existing heterochromatin regions, RITS forms a complex with siRNAs <a href="/wiki/Complementarity_(molecular_biology)" id="w">complementary</a> to the local genes and stably binds local methylated histones, acting co-transcriptionally to degrade any nascent pre-mRNA transcripts that are initiated by <a href="/wiki/RNA_polymerase" id="w">RNA polymerase</a>. The formation of such a heterochromatin region, though not its maintenance, is dicer-dependent, presumably because dicer is required to generate the initial complement of siRNAs that target subsequent transcripts.<sup id="_ref-Noma_a" class="reference"><a href="#_note-Noma" title="">[36]</a></sup> Heterochromatin maintenance has been suggested to function as a self-reinforcing feedback loop, as new siRNAs are formed from the occasional nascent transcripts by RdRP for incorporation into local RITS complexes.<sup id="_ref-Sugiyama_a" class="reference"><a href="#_note-Sugiyama" title="">[37]</a></sup> The relevance of observations from fission yeast mating-type regions and centromeres to <a href="/wiki/Mammal" id="w">mammals</a> is not clear, as heterochromatin maintenance in mammalian cells may be independent of the components of the RNAi pathway.<sup id="_ref-Wang_2006_a" class="reference"><a href="#_note-Wang_2006" title="">[38]</a></sup></p>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Rnai_diagram_retrovirology.png" title="Illustration of the major differences between plant and animal gene silencing. Natively expressed [[microRNA]] or exogenous [[small interfering RNA]] is processed by [[dicer]] and integrated into the [[RISC]] complex, which mediates gene silencing.&#60;ref name=&#34;Saumet&#34;&#62;{{cite journal}}&#60;/ref&#62;"><img src="http://en.wikipedia.org/wiki/300px-Rnai_diagram_retrovirology.png" alt="Illustration of the major differences between plant and animal gene silencing. Natively expressed [[microRNA]] or exogenous [[small interfering RNA]] is processed by [[dicer]] and integrated into the [[RISC]] complex, which mediates gene silencing.&#60;ref name=&#34;Saumet&#34;&#62;{{cite journal}}&#60;/ref&#62;" title="Illustration of the major differences between plant and animal gene silencing. Natively expressed [[microRNA]] or exogenous [[small interfering RNA]] is processed by [[dicer]] and integrated into the [[RISC]] complex, which mediates gene silencing.&#60;ref name=&#34;Saumet&#34;&#62;{{cite journal}}&#60;/ref&#62;" class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Illustration of the major differences between plant and animal gene silencing. Natively expressed <a href="/wiki/MicroRNA" id="w">microRNA</a> or exogenous <a href="/wiki/Small_interfering_RNA" id="w">small interfering RNA</a> is processed by <a href="/wiki/Dicer" id="w">dicer</a> and integrated into the <a href="/wiki/RISC" id="w">RISC</a> complex, which mediates gene silencing.<sup id="_ref-Saumet_a" class="reference"><a href="#_note-Saumet" title="">[39]</a></sup></div></div>
</p>
<a id="Variation_among_organisms" name="Variation_among_organisms"></a><h2>Variation among organisms</h2>
<p>Organisms vary in their ability to take up foreign dsRNA and use it in the RNAi pathway. The effects of RNA interference can be both systemic and heritable in plants and <i><a href="/wiki/C._elegans" id="w">C. elegans</a></i>, although not in <i><a href="/wiki/Drosophila" id="w">Drosophila</a></i> or <a href="/wiki/Mammal" id="w">mammals</a>. In plants, RNAi is thought to propagate by the transfer of siRNAs between cells through <a href="/wiki/Plasmodesmata" id="w">plasmodesmata</a> (channels in the cell walls that enable communication and transport).<sup id="_ref-Lodish_b" class="reference"><a href="#_note-Lodish" title="">[19]</a></sup> The heritability comes from <a href="/wiki/DNA_methylation" id="w">methylation</a> of promoters targeted by RNAi; the new methylation pattern is copied in each new generation of the cell.<sup id="_ref-40" class="reference"><a href="#_note-40" title="">[40]</a></sup> A broad general distinction between plants and animals lies in the targeting of endogenously produced miRNAs; in plants, miRNAs are usually perfectly or nearly perfectly complementary to their target genes and induce direct mRNA cleavage by RISC, while animals&#39; miRNAs tend to be more divergent in sequence and induce translational repression.<sup id="_ref-Saumet_b" class="reference"><a href="#_note-Saumet" title="">[39]</a></sup> This translational effect may be produced by inhibiting the interactions of translation <a href="/wiki/Eukaryotic_initiation_factor" id="w">initiation factors</a> with the messenger RNA&#39;s <a href="/wiki/Polyadenylation" id="w">polyadenine tail</a>.<sup id="_ref-Humphreys_a" class="reference"><a href="#_note-Humphreys" title="">[41]</a></sup></p>
<p>Some eukaryotic protozoa such as <i><a href="/wiki/Leishmania_major" id="w">Leishmania major</a></i> and <i><a href="/wiki/Trypanosoma_cruzi" id="w">Trypanosoma cruzi</a></i> lack the RNAi pathway entirely.<sup id="_ref-DaRocha_2004_a" class="reference"><a href="#_note-DaRocha_2004" title="">[42]</a></sup><sup id="_ref-Robinson_2003_a" class="reference"><a href="#_note-Robinson_2003" title="">[43]</a></sup> Most or all of the components are also missing in some <a href="/wiki/Fungi" id="w">fungi</a>, most notably the <a href="/wiki/Model_organism" id="w">model organism</a> <i><a href="/wiki/Saccharomyces_cerevisiae" id="w">Saccharomyces cerevisiae</a></i>.<sup id="_ref-Aravind_a" class="reference"><a href="#_note-Aravind" title="">[44]</a></sup> Certain <a href="/wiki/Ascomycete" id="w">ascomycetes</a> and <a href="/wiki/Basidiomycete" id="w">basidiomycetes</a> are also missing RNA interference pathways; this observation indicates that proteins required for RNA silencing have been lost independently from many fungal <a href="/wiki/Lineage_(evolution)" id="w">lineages</a>, possibly due to the evolution of a novel pathway with similar function, or to the lack of selective advantage in certain <a href="/wiki/Ecological_niche" id="w">niches</a>.<sup id="_ref-Nakayashiki_a" class="reference"><a href="#_note-Nakayashiki" title="">[45]</a></sup></p>
<a id="Related_prokaryotic_systems" name="Related_prokaryotic_systems"></a><h2>Related prokaryotic systems</h2>
<p>Gene expression in prokaryotes is influenced by an RNA-based system similar in some respects to RNAi. Here, RNA-encoding genes control mRNA abundance or translation by producing a complementary RNA that binds to an mRNA by base pairing. However these regulatory RNAs are not generally considered to be analogous to miRNAs because the dicer enzyme is not involved.<sup id="_ref-Morita_2006_a" class="reference"><a href="#_note-Morita_2006" title="">[46]</a></sup> It has been suggested that <a href="/wiki/CRISPR" id="w">CRISPR</a> systems in prokaryotes are analogous to eukaryotic RNA interference systems, although none of the protein components are <a href="/wiki/Homology_(biology)" id="w">orthologous</a>.<sup id="_ref-makarova_a" class="reference"><a href="#_note-makarova" title="">[47]</a></sup></p>
<a id="Biological_functions" name="Biological_functions"></a><h1>Biological functions</h1>
<a id="Immunity" name="Immunity"></a><h2>Immunity</h2>
<p>RNA interference is a vital part of the <a href="/wiki/Immune_response" id="w">immune response</a> to <a href="/wiki/Virus" id="w">viruses</a> and other foreign <a href="/wiki/Genes" id="w">genetic material</a>, especially in plants where it may also prevent self-propagation by <a href="/wiki/Transposon" id="w">transposons</a>.<sup id="_ref-Stram_2006_a" class="reference"><a href="#_note-Stram_2006" title="">[48]</a></sup> Plants such as <i><a href="/wiki/Arabidopsis_thaliana" id="w">Arabidopsis thaliana</a></i> express multiple dicer <a href="/wiki/Homology_(biology)" id="w">homologs</a> that are specialized to react differently when the plant is exposed to different types of viruses.<sup id="_ref-Blevins_a" class="reference"><a href="#_note-Blevins" title="">[49]</a></sup> Even before the RNAi pathway was fully understood, it was known that induced gene silencing in plants could spread throughout the plant in a systemic effect, and could be transferred from stock to scion plants via <a href="/wiki/Grafting" id="w">grafting</a>.<sup id="_ref-Palauqui_a" class="reference"><a href="#_note-Palauqui" title="">[50]</a></sup> This phenomenon has since been recognized as a feature of the plant adaptive immune system, and allows the entire plant to respond to a virus after an initial localized encounter.<sup id="_ref-Voinnet_a" class="reference"><a href="#_note-Voinnet" title="">[51]</a></sup> In response, many plant viruses have evolved elaborate mechanisms that suppress the RNAi response in plant cells.<sup id="_ref-Lucy_a" class="reference"><a href="#_note-Lucy" title="">[52]</a></sup> These include viral proteins that bind short double-stranded RNA fragments with single-stranded overhang ends, such as those produced by the action of dicer.<sup id="_ref-Merai_a" class="reference"><a href="#_note-Merai" title="">[53]</a></sup> Some plant genomes also express endogenous siRNAs in response to infection by specific types of <a href="/wiki/Bacteria" id="w">bacteria</a>.<sup id="_ref-Katiyar-Agarwal_a" class="reference"><a href="#_note-Katiyar-Agarwal" title="">[54]</a></sup> These effects may be part of a generalized response to pathogens that downregulates any metabolic processes in the host that aid the infection process.<sup id="_ref-Fritz_a" class="reference"><a href="#_note-Fritz" title="">[55]</a></sup></p>
<p>Although animals generally express fewer variants of the dicer enzyme than plants, RNAi in some animals has also been shown to produce an antiviral response. In both juvenile and adult <i>Drosophila</i>, RNA interference is important in antiviral <a href="/wiki/Innate_immunity" id="w">innate immunity</a> and is active against pathogens such as <a href="/wiki/Drosophila_X_virus" id="w">Drosophila X virus</a>.<sup id="_ref-Zambon_a" class="reference"><a href="#_note-Zambon" title="">[56]</a></sup><sup id="_ref-Wang_a" class="reference"><a href="#_note-Wang" title="">[57]</a></sup> A similar role in immunity may operate in <i>C. elegans</i>, as argonaute proteins are upregulated in response to viruses and worms that overexpress components of the RNAi pathway are resistant to viral infection.<sup id="_ref-Lu_a" class="reference"><a href="#_note-Lu" title="">[58]</a></sup><sup id="_ref-Wilkins_a" class="reference"><a href="#_note-Wilkins" title="">[59]</a></sup></p>
<p>The role of RNA interference in mammalian innate immunity is poorly understood, and relatively little data is available. However, the existence of viruses that encode genes able to suppress the RNAi response in mammalian cells may be evidence in favour of an RNAi-dependent mammalian immune response.<sup id="_ref-Berkhout_a" class="reference"><a href="#_note-Berkhout" title="">[60]</a></sup><sup id="_ref-Schutz_a" class="reference"><a href="#_note-Schutz" title="">[61]</a></sup> However, this hypothesis of RNAi-mediated immunity in mammals has been challenged as poorly substantiated.<sup id="_ref-Cullen_a" class="reference"><a href="#_note-Cullen" title="">[62]</a></sup> Alternative functions for RNAi in mammalian viruses also exist, such as miRNAs expressed by the <a href="/wiki/Herpes_virus" id="w">herpes virus</a> that may act as <a href="/wiki/Heterochromatin" id="w">heterochromatin</a> organization triggers to mediate viral latency.<sup id="_ref-Li_b" class="reference"><a href="#_note-Li" title="">[35]</a></sup></p>
<p><a class="internal" href="/wiki/Image:Microrna_secondary_structure.png" title="The [[stem-loop]] [[secondary structure]] of a pre-[[microRNA]] from &#39;&#39;[[Brassica oleracea]]&#39;&#39;."><img src="http://en.wikipedia.org/wiki/Microrna_secondary_structure.png" alt="The [[stem-loop]] [[secondary structure]] of a pre-[[microRNA]] from &#39;&#39;[[Brassica oleracea]]&#39;&#39;." title="The [[stem-loop]] [[secondary structure]] of a pre-[[microRNA]] from &#39;&#39;[[Brassica oleracea]]&#39;&#39;." class="location-right type-thumb" />
</a>
<div class="thumbcaption">The <a href="/wiki/Stem-loop" id="w">stem-loop</a> <a href="/wiki/Secondary_structure" id="w">secondary structure</a> of a pre-<a href="/wiki/MicroRNA" id="w">microRNA</a> from <i><a href="/wiki/Brassica_oleracea" id="w">Brassica oleracea</a></i>.</div>
</p><a id="Downregulation_of_genes" name="Downregulation_of_genes"></a><h2>Downregulation of genes</h2>
<p>Endogenously expressed <a href="/wiki/MiRNA" id="w">miRNAs</a>, including both <a href="/wiki/Intron" id="w">intronic</a> and <a href="/wiki/Intergenic_region" id="w">intergenic</a> miRNAs, are most important in <a href="/wiki/Translation_(genetics)" id="w">translational repression</a><sup id="_ref-Saumet_c" class="reference"><a href="#_note-Saumet" title="">[39]</a></sup> and in the regulation of <a href="/wiki/Developmental_biology" id="w">development</a>, especially the timing of <a href="/wiki/Morphogenesis" id="w">morphogenesis</a> and the maintenance of <a href="/wiki/Cell_differentiation" id="w">undifferentiated</a> or incompletely differentiated cell types such as <a href="/wiki/Stem_cell" id="w">stem cells</a>.<sup id="_ref-Carrington_a" class="reference"><a href="#_note-Carrington" title="">[63]</a></sup> The role of endogenously expressed miRNA in downregulating <a href="/wiki/Gene_expression" id="w">gene expression</a> was first described in <i>C. elegans</i> in 1993.<sup id="_ref-Lee_1993_a" class="reference"><a href="#_note-Lee_1993" title="">[64]</a></sup> In plants this function was discovered when the &#34;JAW microRNA&#34; of <i><a href="/wiki/Arabidopsis_thaliana" id="w">Arabidopsis</a></i> was shown to be involved in the regulation of several genes that control plant shape.<sup id="_ref-Palatnik_2003_a" class="reference"><a href="#_note-Palatnik_2003" title="">[65]</a></sup> In plants, the majority of genes regulated by miRNAs are <a href="/wiki/Transcription_factor" id="w">transcription factors</a>;<sup id="_ref-Zhang_plant_a" class="reference"><a href="#_note-Zhang_plant" title="">[66]</a></sup> thus miRNA activity is particularly wide-ranging and regulates entire <a href="/wiki/Gene_network" id="w">gene networks</a> during development by modulating the expression of key regulatory genes, including transcription factors as well as <a href="/wiki/F-box_protein" id="w">F-box proteins</a>.<sup id="_ref-Jones-Rhoades_a" class="reference"><a href="#_note-Jones-Rhoades" title="">[67]</a></sup> In many organisms, including humans, miRNAs have also been linked to the formation of <a href="/wiki/Tumor" id="w">tumors</a> and dysregulation of the <a href="/wiki/Cell_cycle" id="w">cell cycle</a>. Here, miRNAs can function as both <a href="/wiki/Oncogene" id="w">oncogenes</a> and <a href="/wiki/Tumor_suppressor" id="w">tumor suppressors</a>.<sup id="_ref-Zhang_cancer_a" class="reference"><a href="#_note-Zhang_cancer" title="">[68]</a></sup></p>
<a id="Upregulation_of_genes" name="Upregulation_of_genes"></a><h2>Upregulation of genes</h2>
<p>RNA sequences (siRNA and miRNA) that are complementary to parts of a promoter can increase gene transcription, a phenomenon dubbed <a href="/wiki/RNAa" id="w">RNA activation</a>. As in gene silencing by RNA interference, Dicer and RISC are involved, but the mechanism for how RISC upregulates a gene is unknown.<sup id="_ref-69" class="reference"><a href="#_note-69" title="">[69]</a></sup></p>
<a id="Crosstalk_with_RNA_editing" name="Crosstalk_with_RNA_editing"></a><h2>Crosstalk with RNA editing</h2>
<p>The type of <a href="/wiki/RNA_editing" id="w">RNA editing</a> that is most prevalent in higher eukaryotes converts <a href="/wiki/Adenosine" id="w">adenosine</a> nucleotides into <a href="/wiki/Inosine" id="w">inosine</a> in dsRNAs via the enzyme <a href="/wiki/Adenosine_deaminase" id="w">adenosine deaminase</a> (ADAR).<sup id="_ref-Bass_2002_a" class="reference"><a href="#_note-Bass_2002" title="">[70]</a></sup> It was originally proposed in 2000 that the RNAi and A→I RNA editing pathways might compete for a common dsRNA substrate.<sup id="_ref-Bass_2000_a" class="reference"><a href="#_note-Bass_2000" title="">[71]</a></sup>  Indeed, some pre-miRNAs do undergo A→I RNA editing,<sup id="_ref-Luciano_2004_a" class="reference"><a href="#_note-Luciano_2004" title="">[72]</a></sup><sup id="_ref-yang2006_a" class="reference"><a href="#_note-yang2006" title="">[73]</a></sup> and this mechanism may regulate the processing and expression of mature miRNAs.<sup id="_ref-yang2006_b" class="reference"><a href="#_note-yang2006" title="">[73]</a></sup> Furthermore, at least one mammalian ADAR can sequester siRNAs from RNAi pathway components.<sup id="_ref-Yang_2005_a" class="reference"><a href="#_note-Yang_2005" title="">[74]</a></sup> Further support for this model comes from studies on ADAR-null <i>C. elegans</i> strains indicating that A→I RNA editing may counteract RNAi silencing of endogenous genes and transgenes.<sup id="_ref-Nishikura_2006_a" class="reference"><a href="#_note-Nishikura_2006" title="">[75]</a></sup></p>
<a id="Evolution" name="Evolution"></a><h1>Evolution</h1>
<p>Based on <a href="/wiki/Computational_phylogenetics#maximum parsimony" id="w">parsimony-based</a> <a href="/wiki/Phylogenetic" id="w">phylogenetic</a> analysis, the <a href="/wiki/Most_recent_common_ancestor" id="w">most recent common ancestor</a> of all <a href="/wiki/Eukaryote" id="w">eukaryotes</a> most likely already possessed an early RNA interference pathway; the absence of the pathway in certain eukaryotes is thought to be a derived characteristic. The ancestral RNAi system probably contained at least one <a href="/wiki/Dicer" id="w">dicer</a>-like protein, one <a href="/wiki/Argonaute" id="w">argonaute</a>, one <a href="/wiki/PIWI_protein" id="w">PIWI protein</a>, and an <a href="/wiki/RNA_dependent_RNA_polymerase" id="w">RNA dependent RNA polymerase</a> that may have also played other cellular roles.<sup id="_ref-Cerutti_a" class="reference"><a href="#_note-Cerutti" title="">[76]</a></sup> A large-scale <a href="/wiki/Comparative_genomics" id="w">comparative genomics</a> study likewise indicates that the eukaryotic <a href="/wiki/Crown_group" id="w">crown group</a> already possessed these components, which may then have had closer functional associations with generalized RNA degradation systems such as the <a href="/wiki/Exosome_complex" id="w">exosome</a>.<sup id="_ref-Anantharaman_a" class="reference"><a href="#_note-Anantharaman" title="">[77]</a></sup> This study also suggests that the RNA-binding argonaute protein family, which is shared among eukaryotes, most archaea, and at least some bacteria (such as <i><a href="/wiki/Aquifex_aeolicus" id="w">Aquifex aeolicus</a></i>), is homologous to and originally evolved from components of the <a href="/wiki/Translation_(genetics)" id="w">translation initiation</a> system.</p>
<p>The ancestral function of the RNAi system is generally agreed to have been immune defense against exogenous genetic elements such as <a href="/wiki/Transposon" id="w">transposons</a> and <a href="/wiki/Virus" id="w">viral</a> genomes.<sup id="_ref-Cerutti_b" class="reference"><a href="#_note-Cerutti" title="">[76]</a></sup><sup id="_ref-Buchon_a" class="reference"><a href="#_note-Buchon" title="">[78]</a></sup> Related functions such as <a href="/wiki/Histone" id="w">histone</a> modification may have already been present in the ancestor of modern eukaryotes, although other functions such as regulation of development by miRNA are thought to have evolved later.<sup id="_ref-Cerutti_c" class="reference"><a href="#_note-Cerutti" title="">[76]</a></sup></p>
<p>RNA interference genes, as components of the antiviral innate immune system in many eukaryotes, are involved in an <a href="/wiki/Evolutionary_arms_race" id="w">evolutionary arms race</a> with viral genes. Some viruses have evolved mechanisms for suppressing the RNAi response in their host cells, an effect that has been noted particularly for plant viruses.<sup id="_ref-Lucy_b" class="reference"><a href="#_note-Lucy" title="">[52]</a></sup> Studies of evolutionary rates in <i>Drosophila</i> have shown that genes in the RNAi pathway are subject to strong <a href="/wiki/Directional_selection" id="w">directional selection</a> and are among the fastest-<a href="/wiki/Evolution" id="w">evolving</a> genes in the <i>Drosophila</i> <a href="/wiki/Genome" id="w">genome</a>.<sup id="_ref-Obbard_a" class="reference"><a href="#_note-Obbard" title="">[79]</a></sup></p>
<a id="Technological_applications" name="Technological_applications"></a><h1>Technological applications</h1>
<p><div style="wigth:250px"><a class="internal" href="/wiki/Image:250px-Drosophila_melanogaster_-_side_(aka).jpg" title="A normal adult &#39;&#39;[[Drosophila melanogaster|Drosophila]]&#39;&#39; fly, a common model organism used in RNAi experiments."><img src="http://en.wikipedia.org/wiki/250px-Drosophila_melanogaster_-_side_(aka).jpg" alt="A normal adult &#39;&#39;[[Drosophila melanogaster|Drosophila]]&#39;&#39; fly, a common model organism used in RNAi experiments." title="A normal adult &#39;&#39;[[Drosophila melanogaster|Drosophila]]&#39;&#39; fly, a common model organism used in RNAi experiments." class="location-right type-thumb" width="250px" />
</a>
<div class="thumbcaption">A normal adult <i><a href="/wiki/Drosophila_melanogaster" id="w">Drosophila</a></i> fly, a common model organism used in RNAi experiments.</div></div>

<div style="wigth:250px"><a class="internal" href="/wiki/Image:250px-Celegans_wt_nhr80rnai.png" title="An adult &#39;&#39;[[Caenorhabditis elegans|C. elegans]]&#39;&#39; worm, grown under RNAi suppression of a [[nuclear hormone receptor]] involved in [[desaturase]] regulation. These worms have abnormal [[fatty acid]] metabolism but are viable and fertile.&#60;ref name=Brock_2006&#62;{{cite journal}}&#60;/ref&#62;"><img src="http://en.wikipedia.org/wiki/250px-Celegans_wt_nhr80rnai.png" alt="An adult &#39;&#39;[[Caenorhabditis elegans|C. elegans]]&#39;&#39; worm, grown under RNAi suppression of a [[nuclear hormone receptor]] involved in [[desaturase]] regulation. These worms have abnormal [[fatty acid]] metabolism but are viable and fertile.&#60;ref name=Brock_2006&#62;{{cite journal}}&#60;/ref&#62;" title="An adult &#39;&#39;[[Caenorhabditis elegans|C. elegans]]&#39;&#39; worm, grown under RNAi suppression of a [[nuclear hormone receptor]] involved in [[desaturase]] regulation. These worms have abnormal [[fatty acid]] metabolism but are viable and fertile.&#60;ref name=Brock_2006&#62;{{cite journal}}&#60;/ref&#62;" class="location-right type-thumb" width="250px" />
</a>
<div class="thumbcaption">An adult <i><a href="/wiki/Caenorhabditis_elegans" id="w">C. elegans</a></i> worm, grown under RNAi suppression of a <a href="/wiki/Nuclear_hormone_receptor" id="w">nuclear hormone receptor</a> involved in <a href="/wiki/Desaturase" id="w">desaturase</a> regulation. These worms have abnormal <a href="/wiki/Fatty_acid" id="w">fatty acid</a> metabolism but are viable and fertile.<sup id="_ref-Brock_2006_a" class="reference"><a href="#_note-Brock_2006" title="">[80]</a></sup></div></div>

</p><a id="Gene_knockdown" name="Gene_knockdown"></a><h2>Gene knockdown</h2>
<p>The RNA interference pathway is often exploited in experimental biology to study the function of genes in <a href="/wiki/Cell_culture" id="w">cell culture</a> and <i>in vivo</i> in <a href="/wiki/Model_organism" id="w">model organisms</a>. Double-stranded RNA is synthesized with a sequence complementary to a gene of interest and introduced into a cell or organism, where it is recognized as exogenous genetic material and activates the RNAi pathway. Using this mechanism, researchers can cause a drastic decrease in the expression of a targeted gene. Studying the effects of this decrease can show the physiological role of the gene product. Since RNAi may not totally abolish expression of the gene, this technique is sometimes referred as a &#34;<a href="/wiki/Gene_knockdown" id="w">knockdown</a>&#34;, to distinguish it from &#34;<a href="/wiki/Gene_knockout" id="w">knockout</a>&#34; procedures in which expression of a gene is entirely eliminated. </p>
<p>Extensive efforts in <a href="/wiki/Computational_biology" id="w">computational biology</a> have been directed toward the design of successful dsRNA reagents that maximize gene knockdown but minimize &#34;off-target&#34; effects. Off-target effects arise when an introduced RNA has a base sequence that can pair with and thus reduce the expression of multiple genes at a time. Such problems occur more frequently when the dsRNA contains repetitive sequences. It has been estimated from studying the genomes of <i><a href="/wiki/H._sapiens" id="w">H. sapiens</a></i>, <i><a href="/wiki/C._elegans" id="w">C. elegans</a></i>, and <i><a href="/wiki/S._pombe" id="w">S. pombe</a></i> that about 10% of possible siRNAs will have substantial off-target effects.<sup id="_ref-Qiu_b" class="reference"><a href="#_note-Qiu" title="">[8]</a></sup> A multitude of software tools have been developed implementing <a href="/wiki/Algorithm" id="w">algorithms</a> for the design of general,<sup id="_ref-Naito_ds_a" class="reference"><a href="#_note-Naito_ds" title="">[81]</a></sup><sup id="_ref-Henschel_a" class="reference"><a href="#_note-Henschel" title="">[82]</a></sup> mammal-specific,<sup id="_ref-Naito_si_a" class="reference"><a href="#_note-Naito_si" title="">[83]</a></sup> and virus-specific<sup id="_ref-Naito_virus_a" class="reference"><a href="#_note-Naito_virus" title="">[84]</a></sup> siRNAs that are automatically checked for possible cross-reactivity. </p>
<p>Depending on the organism and experimental system, the exogenous RNA may be a long strand designed to be cleaved by dicer, or short RNAs designed to serve as siRNA substrates. In most <a href="/wiki/Mammal" id="w">mammalian</a> cells, shorter RNAs are used because long double-stranded RNA molecules induce the mammalian <a href="/wiki/Interferon" id="w">interferon</a> response, a form of <a href="/wiki/Innate_immunity" id="w">innate immunity</a> that reacts nonspecifically to foreign genetic material.<sup id="_ref-Reynolds_a" class="reference"><a href="#_note-Reynolds" title="">[85]</a></sup> Mouse <a href="/wiki/Oocyte" id="w">oocytes</a> and cells from early mouse <a href="/wiki/Embryo" id="w">embryos</a> lack this reaction to exogenous dsRNA and are therefore a common model system for studying gene-knockdown effects in mammals.<sup id="_ref-Stein_oocyte_a" class="reference"><a href="#_note-Stein_oocyte" title="">[86]</a></sup> Specialized laboratory techniques have also been developed to improve the utility of RNAi in mammalian systems by avoiding the direct introduction of siRNA, for example, by stable <a href="/wiki/Transfection" id="w">transfection</a> with a <a href="/wiki/Plasmid" id="w">plasmid</a> encoding the appropriate sequence from which siRNAs can be transcribed,<sup id="_ref-Brummelkamp_a" class="reference"><a href="#_note-Brummelkamp" title="">[87]</a></sup> or by more elaborate <a href="/wiki/Lentivirus" id="w">lentiviral</a> vector systems allowing the inducible activation or deactivation of transcription, known as <i>conditional RNAi</i>.<sup id="_ref-Tiscornia_a" class="reference"><a href="#_note-Tiscornia" title="">[88]</a></sup><sup id="_ref-Ventura_a" class="reference"><a href="#_note-Ventura" title="">[89]</a></sup></p>
<a id="Functional_genomics" name="Functional_genomics"></a><h2>Functional genomics</h2>
<p>Most <a href="/wiki/Functional_genomics" id="w">functional genomics</a> applications of RNAi in animals have used <i>C. elegans</i><sup id="_ref-Kamath_a" class="reference"><a href="#_note-Kamath" title="">[90]</a></sup> and <i>Drosophila</i>,<sup id="_ref-Boutros_a" class="reference"><a href="#_note-Boutros" title="">[91]</a></sup> as these are the common <a href="/wiki/Model_organism" id="w">model organisms</a> in which RNAi is most effective. <i>C. elegans</i> is particularly useful for RNAi research for two reasons: firstly, the effects of the gene silencing are generally heritable, and secondly because delivery of the dsRNA is extremely simple. Through a mechanism whose details are poorly understood, bacteria such as <i><a href="/wiki/Escherichia_coli" id="w">E. coli</a></i> that carry the desired dsRNA can be fed to the worms and will transfer their RNA payload to the worm via the intestinal tract. This &#34;delivery by feeding&#34; is just as effective at inducing gene silencing as more costly and time-consuming delivery methods, such as soaking the worms in dsRNA solution and injecting dsRNA into the gonads.<sup id="_ref-Fortunato_2005_a" class="reference"><a href="#_note-Fortunato_2005" title="">[92]</a></sup> Although delivery is more difficult in most other organisms, efforts are also underway to undertake large-scale genomic screening applications in cell culture with mammalian cells.<sup id="_ref-Cullen_genome_a" class="reference"><a href="#_note-Cullen_genome" title="">[93]</a></sup> </p>
<p>Approaches to the design of genome-wide RNAi libraries can require more sophistication than the design of a single siRNA for a defined set of experimental conditions. <a href="/wiki/Artificial_neural_network" id="w">Artificial neural networks</a> are frequently used to design siRNA libraries<sup id="_ref-Huesken_a" class="reference"><a href="#_note-Huesken" title="">[94]</a></sup> and to predict their likely efficiency at gene knockdown.<sup id="_ref-Ge_a" class="reference"><a href="#_note-Ge" title="">[95]</a></sup> Mass genomic screening is widely seen as a promising method for <a href="/wiki/Genome_annotation" id="w">genome annotation</a> and has triggered the development of high-throughput screening methods based on <a href="/wiki/Microarray" id="w">microarrays</a>.<sup id="_ref-Janitz_a" class="reference"><a href="#_note-Janitz" title="">[96]</a></sup><sup id="_ref-Vanhecke_a" class="reference"><a href="#_note-Vanhecke" title="">[97]</a></sup> However, the utility of these screens and the ability of techniques developed on model organisms to generalize to even closely-related species has been questioned, for example from <i>C. elegans</i> to related parasitic nematodes.<sup id="_ref-Geldhof_a" class="reference"><a href="#_note-Geldhof" title="">[98]</a></sup><sup id="_ref-Geldhof2_a" class="reference"><a href="#_note-Geldhof2" title="">[99]</a></sup> </p>
<p>Functional genomics using RNAi is a particularly attractive technique for genomic mapping and annotation in plants because many plants are <a href="/wiki/Polyploid" id="w">polyploid</a>, which presents substantial challenges for more traditional genetic engineering methods. For example, RNAi has been successfully used for functional genomics studies in <a href="/wiki/Triticum_aestivum" id="w">bread wheat</a> (which is hexaploid)<sup id="_ref-Travella_a" class="reference"><a href="#_note-Travella" title="">[100]</a></sup> as well as more common plant model systems <i><a href="/wiki/Arabidopsis_thaliana" id="w">Arabidopsis</a></i> and <a href="/wiki/Maize" id="w">maize</a>.<sup id="_ref-McGinnis_a" class="reference"><a href="#_note-McGinnis" title="">[101]</a></sup></p>
<a id="Medicine" name="Medicine"></a><h2>Medicine</h2>
<p>It may be possible to exploit RNA interference in therapy. Although it is difficult to introduce long dsRNA strands into mammalian cells due to the <a href="/wiki/Interferon" id="w">interferon</a> response, the use of <a href="/wiki/SiRNA" id="w">short interfering RNA</a> mimics has been more successful.<sup id="_ref-Paddison_a" class="reference"><a href="#_note-Paddison" title="">[102]</a></sup> Among the first applications to reach <a href="/wiki/Clinical_trial" id="w">clinical trials</a> were in the treatment of <a href="/wiki/Macular_degeneration" id="w">macular degeneration</a> and <a href="/wiki/Respiratory_syncytial_virus" id="w">respiratory syncytial virus</a>,<sup id="_ref-Sah_a" class="reference"><a href="#_note-Sah" title="">[103]</a></sup> RNAi has also been shown to be effective in the reversal of induced liver failure in mouse models.<sup id="_ref-Zender_a" class="reference"><a href="#_note-Zender" title="">[104]</a></sup></p>
<p>Other proposed clinical uses center on antiviral therapies, including the inhibition of viral gene expression in cancerous cells,<sup id="_ref-Jiang_a" class="reference"><a href="#_note-Jiang" title="">[105]</a></sup> knockdown of host receptors and coreceptors for <a href="/wiki/HIV" id="w">HIV</a>,<sup id="_ref-106" class="reference"><a href="#_note-106" title="">[106]</a></sup> the silencing of <a href="/wiki/Hepatitis_A" id="w">hepatitis A</a><sup id="_ref-Kusov_a" class="reference"><a href="#_note-Kusov" title="">[107]</a></sup> and <a href="/wiki/Hepatitis_B" id="w">hepatitis B</a> genes,<sup id="_ref-Jia_a" class="reference"><a href="#_note-Jia" title="">[108]</a></sup> silencing of <a href="/wiki/Influenza" id="w">influenza</a> gene expression,<sup id="_ref-Li_c" class="reference"><a href="#_note-Li" title="">[35]</a></sup> and inhibition of <a href="/wiki/Measles" id="w">measles</a> viral replication.<sup id="_ref-Hu_a" class="reference"><a href="#_note-Hu" title="">[109]</a></sup> Potential treatments for <a href="/wiki/Neurodegenerative_disease" id="w">neurodegenerative diseases</a> have also been proposed, with particular attention being paid to the polyglutamine diseases such as <a href="/wiki/Huntington%26%2339%3Bs_disease" id="w">Huntington&#39;s disease</a>.<sup id="_ref-Raoul_a" class="reference"><a href="#_note-Raoul" title="">[110]</a></sup> RNA interference is also often seen as a promising way to treat <a href="/wiki/Cancer" id="w">cancer</a> by silencing genes differentially upregulated in <a href="/wiki/Tumor" id="w">tumor</a> cells or genes involved in <a href="/wiki/Cell_division" id="w">cell division</a>.<sup id="_ref-Putral_a" class="reference"><a href="#_note-Putral" title="">[111]</a></sup><sup id="_ref-Izquierdo_a" class="reference"><a href="#_note-Izquierdo" title="">[112]</a></sup> A key area of research in the use of RNAi for clinical applications is the development of a safe delivery method, which to date has involved mainly <a href="/wiki/Viral_vector" id="w">viral vector</a> systems similar to those suggested for <a href="/wiki/Gene_therapy" id="w">gene therapy</a>.<sup id="_ref-Li_delivery_a" class="reference"><a href="#_note-Li_delivery" title="">[113]</a></sup><sup id="_ref-Takeshita_a" class="reference"><a href="#_note-Takeshita" title="">[114]</a></sup></p>
<p>Despite the proliferation of promising cell culture studies for RNAi-based drugs, some concern has been raised regarding the safety of RNA interference, especially the potential for &#34;off-target&#34; effects in which a gene with a coincidentally similar sequence to the targeted gene is also repressed.<sup id="_ref-Tong_a" class="reference"><a href="#_note-Tong" title="">[115]</a></sup> A computational genomics study estimated that the error rate of off-target interactions is about 10%.<sup id="_ref-Qiu_c" class="reference"><a href="#_note-Qiu" title="">[8]</a></sup> One major study of liver disease in mice led to high death rates in the experimental animals, suggested by researchers to be the result of &#34;oversaturation&#34; of the dsRNA pathway.<sup id="_ref-Grimm_a" class="reference"><a href="#_note-Grimm" title="">[116]</a></sup></p>
<a id="Biotechnology" name="Biotechnology"></a><h2>Biotechnology</h2>
<p>RNA interference has been used for applications in <a href="/wiki/Biotechnology" id="w">biotechnology</a>, particularly in the engineering of food plants that produce lower levels of natural plant toxins. Such techniques take advantage of the stable and heritable RNAi phenotype in plant stocks. For example, <a href="/wiki/Cotton" id="w">cotton</a> seeds are rich in <a href="/wiki/Protein_in_nutrition" id="w">dietary protein</a> but naturally contain the toxic <a href="/wiki/Terpenoid" id="w">terpenoid</a> product <a href="/wiki/Gossypol" id="w">gossypol</a>, making them unsuitable for human consumption. RNAi has been used to produce cotton stocks whose seeds contain reduced levels of <a href="/wiki/Delta-cadinene_synthase" id="w">delta-cadinene synthase</a>, a key enzyme in gossypol production, without affecting the enzyme&#39;s production in other parts of the plant, where gossypol is important in preventing damage from plant pests.<sup id="_ref-Sunilkumar_a" class="reference"><a href="#_note-Sunilkumar" title="">[117]</a></sup> Similar efforts have been directed toward the reduction of the <a href="/wiki/Cyanide" id="w">cyanogenic</a> natural product <a href="/wiki/Linamarin" id="w">linamarin</a> in <a href="/wiki/Cassava" id="w">cassava</a> plants.<sup id="_ref-Siritunga_a" class="reference"><a href="#_note-Siritunga" title="">[118]</a></sup> </p>
<p>Although no plant products that use RNAi-based <a href="/wiki/Genetic_engineering" id="w">genetic engineering</a> have yet passed the experimental stage, development efforts have successfully reduced the levels of <a href="/wiki/Allergen" id="w">allergens</a> in <a href="/wiki/Tomato" id="w">tomato</a> plants<sup id="_ref-Le_tomato_a" class="reference"><a href="#_note-Le_tomato" title="">[119]</a></sup> and decreased the precursors of likely <a href="/wiki/Carcinogen" id="w">carcinogens</a> in <a href="/wiki/Tobacco" id="w">tobacco</a> plants.<sup id="_ref-Gavilano_a" class="reference"><a href="#_note-Gavilano" title="">[120]</a></sup> Other plant traits that have been engineered in the laboratory include the production of non-<a href="/wiki/Narcotic" id="w">narcotic</a> natural products by the <a href="/wiki/Opium_poppy" id="w">opium poppy</a>,<sup id="_ref-Allen_a" class="reference"><a href="#_note-Allen" title="">[121]</a></sup> resistance to common plant viruses,<sup id="_ref-Zadeh_a" class="reference"><a href="#_note-Zadeh" title="">[122]</a></sup> and fortification of plants such as tomatoes with dietary <a href="/wiki/Antioxidant" id="w">antioxidants</a>.<sup id="_ref-Niggeweg_a" class="reference"><a href="#_note-Niggeweg" title="">[123]</a></sup> Previous commercial products, including the <a href="/wiki/Flavr_Savr" id="w">Flavr Savr</a> tomato and two <a href="/wiki/Cultivar" id="w">cultivars</a> of <a href="/wiki/Papaya_ringspot_virus" id="w">ringspot</a>-resistant <a href="/wiki/Papaya" id="w">papaya</a>, were originally developed using <a href="/wiki/Antisense" id="w">antisense</a> technology but likely exploited the RNAi pathway.<sup id="_ref-Sanders_a" class="reference"><a href="#_note-Sanders" title="">[124]</a></sup><sup id="_ref-Chiang_a" class="reference"><a href="#_note-Chiang" title="">[125]</a></sup></p>
<a id="History_and_discovery" name="History_and_discovery"></a><h1> History and discovery </h1>
<p><div style="wigth:250px"><a class="internal" href="/wiki/Image:250px-Rnai_phenotype_petunia_crop.png" title="Example [[petunia]] plants in which genes for pigmentation are silenced by RNAi. The left plant is [[wild-type]]; the right plants contain [[transgene]]s that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower.&#60;ref name=&#34;Matzke&#34;&#62;{{cite journal}}&#60;/ref&#62;"><img src="http://en.wikipedia.org/wiki/250px-Rnai_phenotype_petunia_crop.png" alt="Example [[petunia]] plants in which genes for pigmentation are silenced by RNAi. The left plant is [[wild-type]]; the right plants contain [[transgene]]s that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower.&#60;ref name=&#34;Matzke&#34;&#62;{{cite journal}}&#60;/ref&#62;" title="Example [[petunia]] plants in which genes for pigmentation are silenced by RNAi. The left plant is [[wild-type]]; the right plants contain [[transgene]]s that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower.&#60;ref name=&#34;Matzke&#34;&#62;{{cite journal}}&#60;/ref&#62;" class="location-right type-thumb" width="250px" />
</a>
<div class="thumbcaption">Example <a href="/wiki/Petunia" id="w">petunia</a> plants in which genes for pigmentation are silenced by RNAi. The left plant is <a href="/wiki/Wild-type" id="w">wild-type</a>; the right plants contain <a href="/wiki/Transgene" id="w">transgenes</a> that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower.<sup id="_ref-Matzke_a" class="reference"><a href="#_note-Matzke" title="">[126]</a></sup></div></div>

<a class="internal" href="/wiki/Image:Craig_mello.jpg" title="[[Craig Mello]] at the 2006 [[Nobel Prize]] lecture."><img src="http://en.wikipedia.org/wiki/Craig_mello.jpg" alt="[[Craig Mello]] at the 2006 [[Nobel Prize]] lecture." title="[[Craig Mello]] at the 2006 [[Nobel Prize]] lecture." class="location-right type-thumb" />
</a>
<div class="thumbcaption"><a href="/wiki/Craig_Mello" id="w">Craig Mello</a> at the 2006 <a href="/wiki/Nobel_Prize" id="w">Nobel Prize</a> lecture.</div>
The discovery of RNAi was preceded first by observations of transcriptional inhibition by <a href="/wiki/Antisense" id="w">antisense</a> RNA expressed in <a href="/wiki/Transgenic" id="w">transgenic</a> plants,<sup id="_ref-Ecker_1986_a" class="reference"><a href="#_note-Ecker_1986" title="">[127]</a></sup> and more directly by reports of unexpected outcomes in experiments performed by plant scientists in the <a href="/wiki/United_States" id="w">U.S.</a> and <a href="/wiki/The_Netherlands" id="w">The Netherlands</a> in the early 1990s.<sup id="_ref-Napoli_1990_a" class="reference"><a href="#_note-Napoli_1990" title="">[128]</a></sup> In an attempt to alter <a href="/wiki/Flower" id="w">flower</a> colors in <a href="/wiki/Petunia" id="w">petunias</a>, researchers introduced additional copies of a gene encoding <a href="/wiki/Chalcone_synthase" id="w">chalcone synthase</a>, a key enzyme for flower <a href="/wiki/Pigmentation" id="w">pigmentation</a> into petunia plants of normally pink or violet flower color. The overexpressed gene was expected to result in darker flowers, but instead produced less pigmented, fully or partially white flowers, indicating that the activity of chalcone synthase had been substantially decreased; in fact, both the endogenous genes and the transgenes were downregulated in the white flowers. Soon after, a related event termed <i>quelling</i> was noted in the <a href="/wiki/Fungus" id="w">fungus</a> <i><a href="/wiki/Neurospora_crassa" id="w">Neurospora crassa</a></i>,<sup id="_ref-Romano_a" class="reference"><a href="#_note-Romano" title="">[129]</a></sup> although it was not immediately recognized as related. Further investigation of the phenomenon in plants indicated that the downregulation was due to post-transcriptional inhibition of gene expression via an increased rate of mRNA degradation.<sup id="_ref-Van_Blokland_1994_a" class="reference"><a href="#_note-Van_Blokland_1994" title="">[130]</a></sup> This phenomenon was called <i>co-suppression of gene expression</i>, but the molecular mechanism remained unknown.</p>
<p>Not long after, plant <a href="/wiki/Virologist" id="w">virologists</a> working on improving plant resistance to viral diseases observed a similar unexpected phenomenon. While it was known that plants expressing virus-specific proteins showed enhanced tolerance or resistance to viral infection, it was not expected that plants carrying only short, non-coding regions of viral RNA sequences would show similar levels of protection. Researchers believed that viral RNA produced by transgenes could also inhibit viral replication.<sup id="_ref-Covey_1997_a" class="reference"><a href="#_note-Covey_1997" title="">[131]</a></sup> The reverse experiment, in which short sequences of plant genes were introduced into viruses, showed that the targeted gene was suppressed in an infected plant. This phenomenon was labeled &#34;virus-induced gene silencing&#34; (VIGS), and the set of such phenomena were collectively called <a href="/wiki/Post_transcriptional_gene_silencing" id="w">post transcriptional gene silencing</a>.<sup id="_ref-Ratcliff_1997_a" class="reference"><a href="#_note-Ratcliff_1997" title="">[132]</a></sup></p>
<p>After these initial observations in plants, many laboratories around the world searched for the occurrence of this phenomenon in other organisms.<sup id="_ref-Guo_a" class="reference"><a href="#_note-Guo" title="">[133]</a></sup><sup id="_ref-Pal-Bhadra_a" class="reference"><a href="#_note-Pal-Bhadra" title="">[134]</a></sup> <a href="/wiki/Craig_C._Mello" id="w">Craig C. Mello</a> and <a href="/wiki/Andrew_Fire" id="w">Andrew Fire</a>&#39;s 1998 <i>Nature</i> paper reported a potent gene silencing effect after injecting double stranded RNA into <i><a href="/wiki/C._elegans" id="w">C. elegans</a></i>.<sup id="_ref-Fire_b" class="reference"><a href="#_note-Fire" title="">[3]</a></sup> In investigating the regulation of muscle protein production, they observed that neither mRNA nor <a href="/wiki/Antisense_RNA" id="w">antisense RNA</a> injections had an effect on protein production, but double-stranded RNA successfully silenced the targeted gene. As a result of this work, they coined the term <i>RNAi</i>.  Fire and Mello&#39;s discovery was particularly notable because it represented the first identification of the causative agent for the phenomenon. Fire and Mello were awarded the <a href="/wiki/Nobel_Prize_in_Physiology_or_Medicine" id="w">Nobel Prize in Physiology or Medicine</a> in 2006 for their work.<sup id="_ref-Daneholt2006_b" class="reference"><a href="#_note-Daneholt2006" title="">[2]</a></sup></p>
<a id="References" name="References"></a><h1>References</h1>
<p>{{reflist}}</p>
<a id="External_links" name="External_links"></a><h1>External links</h1>
<p>{{Wikiversity}}
</p>
<ul>
<li><a class="externallink" href="http://www.nature.com/focus/rnai/animations/animation/animation.htm" rel="nofollow" title="http://www.nature.com/focus/rnai/animations/animation/animation.htm">Animation of the RNAi process</a>, from <i><a href="/wiki/Nature_(journal)" id="w">Nature</a></i></li>
<li><a class="externallink" href="http://www.pbs.org/wgbh/nova/sciencenow/3210/02.html" rel="nofollow" title="http://www.pbs.org/wgbh/nova/sciencenow/3210/02.html">NOVA scienceNOW explains RNAi</a> - A 15 minute video of the <i><a href="/wiki/Nova_(TV_series)" id="w">Nova</a></i> broadcast that aired on <a href="/wiki/PBS" id="w">PBS</a>, <a href="/wiki/July_26" id="w">July 26</a>, <a href="/wiki/2005" id="w">2005</a></li>
<li><a class="externallink" href="http://www.si-rna.com" rel="nofollow" title="http://www.si-rna.com">RNA interference Database</a></li>
<li>{{PLoSlink}}</li>
<li><a class="externallink" href="http://www.silencinggenomes.org" rel="nofollow" title="http://www.silencinggenomes.org">Silencing Genomes</a> RNA interference (RNAi) experiments and bioinformatics in C. elegans for education. From the Dolan DNA Learning Center of Cold Spring Harbor Laboratory.</li>
<li><a class="externallink" href="http://www.natureprotocols.com/2006/07/19/rnai_screens_in_c_elegans_in_a.php" rel="nofollow" title="http://www.natureprotocols.com/2006/07/19/rnai_screens_in_c_elegans_in_a.php">RNAi screens in C. elegans in a 96-well liquid format and their application to the systematic identification of genetic interactions (a protocol)</a></li>
<li><a class="externallink" href="http://www.nytimes.com/2006/10/03/science/03nobel.html?ex=1180065600&#38;en=e26ec371cb398ea0&#38;ei=5070" rel="nofollow" title="http://www.nytimes.com/2006/10/03/science/03nobel.html?ex=1180065600&#38;en=e26ec371cb398ea0&#38;ei=5070">2 American ‘Worm People’ Win Nobel for RNA Work</a>, from <a href="/wiki/The_New_York_Times" id="w">NY Times</a></li>
<li><a class="externallink" href="http://www.cshprotocols.org/cgi/content/full/2006/21/pdb.prot4511" rel="nofollow" title="http://www.cshprotocols.org/cgi/content/full/2006/21/pdb.prot4511"> Microinjection of dsRNA into Mouse Oocytes and Early Embryos</a></li></ul>
<p>{{featured article}}</p>
<p>


</p>
<p><a href="http://zh-min-nan.wikipedia.org/wiki/RNA_kan-g%C4%81i">zh-min-nan:RNA kan-gāi</a>
<a href="http://cs.wikipedia.org/wiki/RNAi">cs:RNAi</a>
<a href="http://de.wikipedia.org/wiki/RNA-Interferenz">de:RNA-Interferenz</a>
<a href="http://el.wikipedia.org/wiki/Ε%CF%80%CE%B5%CE%BC%CE%B2%CE%B1%CF%84%CE%B9%CE%BA%CF%8C_RNA">el:Επεμβατικό RNA</a>
<a href="http://es.wikipedia.org/wiki/ARN_interferente">es:ARN interferente</a>
<a href="http://fr.wikipedia.org/wiki/Interf%C3%A9rence_ARN">fr:Interférence ARN</a>
<a href="http://it.wikipedia.org/wiki/RNA_interference">it:RNA interference</a>
<a href="http://he.wikipedia.org/wiki/RNAi">he:RNAi</a>
<a href="http://nl.wikipedia.org/wiki/RNA-interferentie">nl:RNA-interferentie</a>
<a href="http://ja.wikipedia.org/wiki/RNAi">ja:RNAi</a>
<a href="http://no.wikipedia.org/wiki/RNA-interferens">no:RNA-interferens</a>
<a href="http://pl.wikipedia.org/wiki/Interferencja_RNA">pl:Interferencja RNA</a>
<a href="http://pt.wikipedia.org/wiki/Rna_de_Interfer%C3%AAncia">pt:Rna de Interferência</a>
<a href="http://ru.wikipedia.org/wiki/Р%D0%9D%D0%9A_%D0%B8%D0%BD%D1%82%D0%B5%D1%80%D1%84%D0%B5%D1%80%D0%B5%D0%BD%D1%86%D0%B8%D1%8F">ru:РНК интерференция</a>
<a href="http://fi.wikipedia.org/wiki/RNA-interferenssi">fi:RNA-interferenssi</a>
<a href="http://sv.wikipedia.org/wiki/RNA-interferens">sv:RNA-interferens</a>
<a href="http://th.wikipedia.org/wiki/RNAi">th:RNAi</a>
<a href="http://ur.wikipedia.org/wiki/ا%D8%B1%D9%86%D8%A7_%D8%AA%D8%AF%D8%A7%D8%AE%D9%84">ur:ارنا تداخل</a>
<a href="http://zh.wikipedia.org/wiki/RNA%E5%B9%B2%E6%89%B0">zh:RNA干扰</a>
</p>