
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-EcDHFR_raytraced.png" title="[[Dihydrofolate reductase]] from &#39;&#39;[[Escherichia coli|E. coli]]&#39;&#39; with its two substrates, [[Dihydrofolic acid|dihydrofolate]] (right) and [[Nicotinamide adenine dinucleotide phosphate|NADPH]] (left), bound in the active site. The protein is shown as a [[ribbon diagram]], with alpha helices in red, beta sheets in yellow and loops in blue. Generated from [http://www.rcsb.org/pdb/explore.do?structureId=7DFR 7DFR]."><img src="http://en.wikipedia.org/wiki/300px-EcDHFR_raytraced.png" alt="[[Dihydrofolate reductase]] from &#39;&#39;[[Escherichia coli|E. coli]]&#39;&#39; with its two substrates, [[Dihydrofolic acid|dihydrofolate]] (right) and [[Nicotinamide adenine dinucleotide phosphate|NADPH]] (left), bound in the active site. The protein is shown as a [[ribbon diagram]], with alpha helices in red, beta sheets in yellow and loops in blue. Generated from [http://www.rcsb.org/pdb/explore.do?structureId=7DFR 7DFR]." title="[[Dihydrofolate reductase]] from &#39;&#39;[[Escherichia coli|E. coli]]&#39;&#39; with its two substrates, [[Dihydrofolic acid|dihydrofolate]] (right) and [[Nicotinamide adenine dinucleotide phosphate|NADPH]] (left), bound in the active site. The protein is shown as a [[ribbon diagram]], with alpha helices in red, beta sheets in yellow and loops in blue. Generated from [http://www.rcsb.org/pdb/explore.do?structureId=7DFR 7DFR]." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption"><a href="/wiki/Dihydrofolate_reductase" id="w">Dihydrofolate reductase</a> from <i><a href="/wiki/Escherichia_coli" id="w">E. coli</a></i> with its two substrates, <a href="/wiki/Dihydrofolic_acid" id="w">dihydrofolate</a> (right) and <a href="/wiki/Nicotinamide_adenine_dinucleotide_phosphate" id="w">NADPH</a> (left), bound in the active site. The protein is shown as a <a href="/wiki/Ribbon_diagram" id="w">ribbon diagram</a>, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from <a class="externallink" href="http://www.rcsb.org/pdb/explore.do?structureId=7DFR" rel="nofollow" title="http://www.rcsb.org/pdb/explore.do?structureId=7DFR">7DFR</a>.</div></div>
</p>
<p><b>Enzyme kinetics</b> is the study of the <a href="/wiki/Chemical_reaction" id="w">chemical reactions</a> that are <a href="/wiki/Catalyst" id="w">catalysed</a> by <a href="/wiki/Enzymes" id="w">enzymes</a>, with a focus on their <a href="/wiki/Reaction_rate" id="w">reaction rates</a>.  The study of an enzyme&#39;s <a href="/wiki/Chemical_kinetics" id="w">kinetics</a> provides insights into the catalytic mechanism of this enzyme, its role in <a href="/wiki/Metabolism" id="w">metabolism</a>, how its activity is controlled in the <a href="/wiki/Cell_(biology)" id="w">cell</a>, and how <a href="/wiki/Drug" id="w">drugs</a> and <a href="/wiki/Poison" id="w">poisons</a> can <a href="/wiki/Enzyme_inhibitor" id="w">inhibit</a> its activity.</p>
<p>Enzymes are <a href="/wiki/Protein" id="w">protein</a> <a href="/wiki/Molecule" id="w">molecules</a> that manipulate other molecules — the enzymes&#39; <a href="/wiki/Substrate_(biochemistry)" id="w">substrates</a>. These target molecules bind to an enzyme&#39;s <a href="/wiki/Active_site" id="w">active site</a> and are transformed into products through a series of steps known as the <a href="/wiki/Enzyme_catalysis" id="w">enzymatic mechanism</a>. These mechanisms can be divided into single-substrate and multiple-substrate mechanisms. 
Kinetic studies on enzymes that only bind one substrate, such as <a href="/wiki/Triosephosphateisomerase" id="w">triosephosphate isomerase</a>, aim to measure the <a href="/wiki/Dissociation_constant" id="w">affinity</a> with which the enzyme binds this substrate and the turnover rate. </p>
<p>When enzymes bind multiple substrates, such as <a href="/wiki/Dihydrofolate_reductase" id="w">dihydrofolate reductase</a> (shown right), enzyme kinetics can also show the sequence in which these substrates bind and the sequence in which products are released.  An example of enzymes that bind a single substrate and release multiple products are <a href="/wiki/Protease" id="w">proteases</a>, which cleave one protein substrate into two polypeptide products. Others join two substrates together, such as <a href="/wiki/DNA_polymerase" id="w">DNA polymerase</a> linking a <a href="/wiki/Nucleotide" id="w">nucleotide</a> to <a href="/wiki/DNA" id="w">DNA</a>. Although these mechanisms are often a complex series of steps, there is typically one <i>rate-determining step</i> that determines the overall kinetics.  This <a href="/wiki/Rate-determining_step" id="w">rate-determining step</a> may be a chemical reaction or a <a href="/wiki/Conformational_isomerism" id="w">conformational</a> change of the enzyme or substrates, such as those involved in the release of product(s) from the enzyme.</p>
<p>Knowledge of the <a href="/wiki/Protein_structure" id="w">enzyme&#39;s structure</a> is helpful in interpreting the kinetic data.  For example, the structure can suggest how substrates and products bind during catalysis; what changes occur during the reaction; and even the role of particular <a href="/wiki/Amino_acid" id="w">amino acid</a> residues in the mechanism.  Some enzymes change shape significantly during the mechanism; in such cases, it is helpful to determine the enzyme structure with and without bound substrate analogs that do not undergo the enzymatic reaction.</p>
<p>Not all biological catalysts are protein enzymes; <a href="/wiki/RNA" id="w">RNA</a>-based catalysts such as <a href="/wiki/Ribozymes" id="w">ribozymes</a> and <a href="/wiki/Ribosomes" id="w">ribosomes</a> are essential to many cellular functions, such as <a href="/wiki/Splicing_(genetics)" id="w">RNA splicing</a> and <a href="/wiki/Translation" id="w">translation</a>. The main difference between ribozymes and enzymes is that the RNA catalysts perform a more limited set of reactions, although their <a href="/wiki/Reaction_mechanism" id="w">reaction mechanisms</a> and kinetics can be analysed and classified by the same methods.</p>
<a id="General_principles" name="General_principles"></a><h1>General principles</h1>
<p><div style="wigth:400px"><a class="internal" href="/wiki/Image:400px-KinEnzymo(en).svg.png" title="The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate."><img src="http://en.wikipedia.org/wiki/400px-KinEnzymo(en).svg.png" alt="The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate." title="The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate." class="location-right type-thumb" width="400px" />
</a>
<div class="thumbcaption">The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate.</div></div>
</p>
<p>The reaction catalysed by an enzyme uses exactly the same reactants and produces exactly the same products as the uncatalysed reaction. Like other <a href="/wiki/Catalysts" id="w">catalysts</a>, enzymes do not alter the position of <a href="/wiki/Chemical_equilibrium" id="w">equilibrium</a> between substrates and products.<sup id="_ref-1" class="reference"><a href="#_note-1" title="">[1]</a></sup> However, unlike normal chemical reactions, enzymes are saturable. This means as more substrate is added, the reaction rate will increase, because more active sites become occupied. This can continue until all the enzyme becomes saturated with substrate and the rate reaches a maximum. The two most important kinetic properties of an enzyme are how quickly the enzyme becomes saturated with a particular substrate, and the maximum rate it can achieve. Knowing these properties suggests what an enzyme might do in the environment of the cell and can show how the enzyme will respond to changes in these conditions.</p>
<a id="Enzyme_assays" name="Enzyme_assays"></a><h1> Enzyme assays </h1>
<p>{{main}}
<div style="wigth:250px"><a class="internal" href="/wiki/Image:250px-Enzyme_progress_curve.svg.png" title="Progress curve for an enzyme reaction. The slope in the initial rate period is the &#39;&#39;&#39;initial rate of reaction&#39;&#39;&#39; &#39;&#39;v&#39;&#39;.  The [[Michaelis-Menten kinetics|Michaelis-Menten equation]] describe how this slope varies with the concentration of substrate."><img src="http://en.wikipedia.org/wiki/250px-Enzyme_progress_curve.svg.png" alt="Progress curve for an enzyme reaction. The slope in the initial rate period is the &#39;&#39;&#39;initial rate of reaction&#39;&#39;&#39; &#39;&#39;v&#39;&#39;.  The [[Michaelis-Menten kinetics|Michaelis-Menten equation]] describe how this slope varies with the concentration of substrate." title="Progress curve for an enzyme reaction. The slope in the initial rate period is the &#39;&#39;&#39;initial rate of reaction&#39;&#39;&#39; &#39;&#39;v&#39;&#39;.  The [[Michaelis-Menten kinetics|Michaelis-Menten equation]] describe how this slope varies with the concentration of substrate." class="location-left type-thumb" width="250px" />
</a>
<div class="thumbcaption">Progress curve for an enzyme reaction. The slope in the initial rate period is the <b>initial rate of reaction</b> <i>v</i>.  The <a href="/wiki/Michaelis-Menten_kinetics" id="w">Michaelis-Menten equation</a> describe how this slope varies with the concentration of substrate.</div></div>

<a href="/wiki/Enzyme_assay" id="w">Enzyme assays</a> are laboratory procedures that measure the rate of enzyme reactions. Because enzymes are not consumed by the reactions they catalyse, enzyme assays usually follow changes in the concentration of either substrates or products to measure the rate of reaction. There are many methods of measurement. <a href="/wiki/Ultraviolet-visible_spectroscopy" id="w">Spectrophotometric</a> assays observe change in the <a href="/wiki/Absorbance" id="w">absorbance</a> of light between products and reactants; radiometric assays involve the incorporation or release of <a href="/wiki/Radioactivity" id="w">radioactivity</a> to measure the amount of product made over time. Spectrophotometric assays are most convenient since they allow the rate of the reaction to be measured continuously. Although radiometric assays require the removal and counting of samples (i.e., they are discontinuous assays) they are usually extremely sensitive and can measure very low levels of enzyme activity.<sup id="_ref-2" class="reference"><a href="#_note-2" title="">[2]</a></sup>  An analogous approach is to use <a href="/wiki/Mass_spectrometry" id="w">mass spectrometry</a> to monitor the incorporation or release of <a href="/wiki/Stable_isotope" id="w">stable isotopes</a> as substrate is converted into product.</p>
<p>The most sensitive enzyme assays use <a href="/wiki/Laser" id="w">lasers</a> focused through a <a href="/wiki/Microscope" id="w">microscope</a> to observe changes in single enzyme molecules as they catalyse their reactions. These measurements either use changes in the <a href="/wiki/Fluorescence" id="w">fluorescence</a> of <a href="/wiki/Cofactor_(biochemistry)" id="w">cofactors</a> during an enzyme&#39;s reaction mechanism, or of <a href="/wiki/Fluorescent_dyes" id="w">fluorescent dyes</a> added onto specific sites of the <a href="/wiki/Protein" id="w">protein</a> to report movements that occur during catalysis.<sup id="_ref-3" class="reference"><a href="#_note-3" title="">[3]</a></sup> These studies are providing a new view of the kinetics and dynamics of single enzymes, as opposed to traditional enzyme kinetics, which observes the average behaviour of populations of millions of enzyme molecules.<sup id="_ref-4" class="reference"><a href="#_note-4" title="">[4]</a></sup><sup id="_ref-5" class="reference"><a href="#_note-5" title="">[5]</a></sup></p>
<p>On the left is shown a typical progress curve for an enzyme assay. The enzyme produces product at a <a href="/wiki/Linear_equation" id="w">linear</a> initial rate at the start of the reaction. Later in this progress curve, the rate slows down as substrate is used up or products accumulate. The length of the initial rate period depends on the assay conditions and can range from milliseconds to hours. Enzyme assays are usually set up to produce an initial rate lasting over a minute, to make measurements easier. However, equipment for rapidly mixing liquids allows fast kinetic measurements on initial rates of less than one second.<sup id="_ref-6" class="reference"><a href="#_note-6" title="">[6]</a></sup> These very rapid assays are essential for measuring pre-steady-state kinetics, which are discussed below.</p>
<p>Most enzyme kinetics studies concentrate on this initial, linear part of enzyme reactions. However, it is also possible to measure the complete reaction curve and fit this data to a non-linear <a href="/wiki/Rate_equation" id="w">rate equation</a>. This way of measuring enzyme reactions is called progress-curve analysis.<sup id="_ref-7" class="reference"><a href="#_note-7" title="">[7]</a></sup> This approach is useful as an alternative to rapid kinetics when the initial rate is too fast to measure accurately.</p>
<a id="Single-substrate_reactions" name="Single-substrate_reactions"></a><h1> Single-substrate reactions </h1>

<p>Enzymes with single-substrate mechanisms include <a href="/wiki/Isomerase" id="w">isomerases</a> such as <a href="/wiki/Triosephosphateisomerase" id="w">triosephosphateisomerase</a> or <a href="/wiki/Bisphosphoglycerate_mutase" id="w">bisphosphoglycerate mutase</a>, intramolecular <a href="/wiki/Lyase" id="w">lyases</a> such as <a href="/wiki/Adenylate_cyclase" id="w">adenylate cyclase</a> and the <a href="/wiki/Ribozyme" id="w">hammerhead ribozyme</a>, a RNA lyase.<sup id="_ref-8" class="reference"><a href="#_note-8" title="">[8]</a></sup> However, some enzymes that only have a single substrate do not fall into this category of mechanisms. <a href="/wiki/Catalase" id="w">Catalase</a> is an example of this, as the enzyme reacts with a first molecule of <a href="/wiki/Hydrogen_peroxide" id="w">hydrogen peroxide</a> substrate, becomes oxidised and is then reduced by a second molecule of substrate. Although a single substrate is involved, the existence of a modified enzyme intermediate means that the mechanism of catalase is actually a ping–pong mechanism, a type of mechanism that is discussed in the <i>Multi-substrate reactions</i> section below.</p>
<a id="Michaelis%E2%80%93Menten_kinetics" name="Michaelis%E2%80%93Menten_kinetics"></a><h2>Michaelis–Menten kinetics</h2>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Michaelis-Menten_saturation_curve_of_an_enzyme_reaction.svg.png" title="Saturation curve for an enzyme showing the relation between the concentration of substrate and rate."><img src="http://en.wikipedia.org/wiki/300px-Michaelis-Menten_saturation_curve_of_an_enzyme_reaction.svg.png" alt="Saturation curve for an enzyme showing the relation between the concentration of substrate and rate." title="Saturation curve for an enzyme showing the relation between the concentration of substrate and rate." class="location-none type-thumb" width="300px" />
</a>
<div class="thumbcaption">Saturation curve for an enzyme showing the relation between the concentration of substrate and rate.</div></div>

<div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Mechanism_plus_rates.svg.png" title="Single-substrate mechanism for an enzyme reaction. &#39;&#39;k&#39;&#39;&#60;sub&#62;1&#60;/sub&#62;, &#39;&#39;k&#39;&#39;&#60;sub&#62;-1&#60;/sub&#62; and &#39;&#39;k&#39;&#39;&#60;sub&#62;2&#60;/sub&#62; are the rate constants for the individual steps."><img src="http://en.wikipedia.org/wiki/300px-Mechanism_plus_rates.svg.png" alt="Single-substrate mechanism for an enzyme reaction. &#39;&#39;k&#39;&#39;&#60;sub&#62;1&#60;/sub&#62;, &#39;&#39;k&#39;&#39;&#60;sub&#62;-1&#60;/sub&#62; and &#39;&#39;k&#39;&#39;&#60;sub&#62;2&#60;/sub&#62; are the rate constants for the individual steps." title="Single-substrate mechanism for an enzyme reaction. &#39;&#39;k&#39;&#39;&#60;sub&#62;1&#60;/sub&#62;, &#39;&#39;k&#39;&#39;&#60;sub&#62;-1&#60;/sub&#62; and &#39;&#39;k&#39;&#39;&#60;sub&#62;2&#60;/sub&#62; are the rate constants for the individual steps." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Single-substrate mechanism for an enzyme reaction. <i>k</i><sub>1</sub>, <i>k</i><sub>-1</sub> and <i>k</i><sub>2</sub> are the rate constants for the individual steps.</div></div>
</p>
<p>As enzyme-catalysed reactions are saturable, their rate of catalysis does not show a linear response to increasing substrate. If the initial rate of the reaction is measured over a range of substrate concentrations (denoted as [S]), the reaction rate (<i>v</i>) increases as [S] increases, as shown on the right. However, as [S] gets higher, the enzyme becomes saturated with substrate and the rate reaches <i>V</i><sub>max</sub>, the enzyme&#39;s maximum rate.</p>
<p>The <a href="/wiki/Michaelis-Menten_kinetics" id="w">Michaelis-Menten kinetic model of a single-substrate reaction</a> is shown on the right. There is an initial <a href="/wiki/Chemical_kinetics" id="w">bimolecular reaction</a> between the enzyme E and substrate S to form the enzyme–substrate complex ES.  Although the enzymatic mechanism for the <a href="/wiki/Chemical_kinetics" id="w">unimolecular reaction</a> <span class="math">ES \rightarrow E + P </span> reaction can be quite complex, there is typically one rate-determining enzymatic step that allows the mechanism to be modelled as a single catalytic step of rate constant <i>k</i><sub>2</sub>.</p>
<dl><dd><span class="math">\begin{matrix}v = k_2 [\mbox{ES}]\end{matrix}</span>&#160;&#160;&#160;&#160;<i>(Equation 1).</i></dd></dl>
<p><i>k</i><sub>2</sub> is also called <i>k</i><sub>cat</sub> or the turnover number, the maximum number of enzymatic reactions catalyzed per second.  </p>
<p>At low concentrations of substrate [S], the enzyme exists in an equilibrium between both the free form E and the enzyme–substrate complex ES; increasing [S] likewise increases [ES] at the expense of [E], shifting the binding equilibrium to the right.  Since the rate of the reaction depends on the concentration [ES], the rate is  sensitive to small changes in [S].  However, at very high [S], the enzyme is entirely saturated with substrate, and exists only in the ES form. Under these conditions, the rate (<i>v</i>≈<i>k</i><sub>2</sub>[E]<sub>tot</sub>=<i>V</i><sub>max</sub>) is insensitive to small changes in [S]; here, [E]<sub>tot</sub> is the total enzyme concentration </p>
<dl><dd><span class="math">
[\mbox{E}]_\mathrm{tot} \ \stackrel{\mathrm{def}}{=}\  [\mbox{E}] + [\mbox{ES}]
</span></dd></dl>
<p>which is approximately equal to the concentration [ES] under saturating conditions.</p>
<p>The <a href="/wiki/Michaelis-Menten_kinetics" id="w">Michaelis–Menten equation</a><sup id="_ref-9" class="reference"><a href="#_note-9" title="">[9]</a></sup> describes how the reaction rate <i>v</i> depends on the position of the substrate-binding <a href="/wiki/Chemical_equilibrium" id="w">equilibrium</a> and the rate constant <i>k</i><sub>2</sub>. <a href="/wiki/Leonor_Michaelis" id="w">Michaelis</a> and <a href="/wiki/Maud_Menten" id="w">Menten</a> showed when <i>k</i><sub>2</sub> is much less than <i>k</i><sub>-1</sub> (called the <i>equilibrium assumption</i>)<sup id="_ref-Cha_a" class="reference"><a href="#_note-Cha" title="">[10]</a></sup> they could derive the following equation:</p>
<dl><dd><span class="math"> v = \frac{V_\max[\mbox{S}]}{K_m + [\mbox{S}]} </span>&#160;&#160;&#160;&#160;<i>(Equation 2)</i></dd></dl>
<p>This Michaelis-Menten equation is the basis for most single-substrate enzyme kinetics.</p>
<p>The Michaelis constant <i>K</i><sub>m</sub> is defined as the concentration at which the rate of the enzyme reaction is half <i>V</i><sub>max</sub>. This may be verified by substituting [S] = <i>K</i><sub>m</sub> into the Michaelis-Menten equation. If the rate-determining enzymatic step is slow compared to substrate dissociation (<i>k</i><sub>2</sub> &#60;&#60; <i>k</i><sub>-1</sub>), the Michaelis constant <i>K</i><sub>m</sub> is roughly the <a href="/wiki/Dissociation_constant" id="w">dissociation constant</a> of the ES complex, although this situation is relatively rare.</p>
<p>The more normal situation where <i>k</i><sub>2</sub> &#62; <i>k</i><sub>-1</sub> is sometimes called Briggs-<a href="/wiki/J._B._S._Haldane" id="w">Haldane</a> kinetics.<sup id="_ref-11" class="reference"><a href="#_note-11" title="">[11]</a></sup> The Michaelis–Menten equation still holds under these more general conditions, as may be derived from the <a href="/wiki/Steady_state_(chemistry)" id="w">steady-state approximation</a>.<sup id="_ref-Cha_b" class="reference"><a href="#_note-Cha" title="">[10]</a></sup> During the initial-rate period, the reaction rate <i>v</i> is roughly constant, indicating that [ES] is similarly constant (cf. equation 1):</p>
<dl><dd><span class="math">
\frac{d}{dt}[\mbox{ES}] = k_{1} [\mbox{E}][\mbox{S}] - k_{2}[\mbox{ES}] - k_{-1}[\mbox{ES}] \approx 0
</span></dd></dl>
<p>Therefore, the concentration [ES] is given by the formula</p>
<dl><dd><span class="math">
[\mbox{ES}] \approx \frac{[\mbox{E}]_\mathrm{tot} [\mbox{S}]}{[\mbox{S}] + K_{m}}
</span></dd></dl>
<p>where the Michaelis constant <i>K</i><sub>m</sub> is defined </p>
<dl><dd><span class="math">
K_{m} \ \stackrel{\mathrm{def}}{=}\  \frac{k_{2} + k_{-1}}{k_{1}} \approx \frac{[\mbox{E}][\mbox{S}]}{[\mbox{ES}]}
</span></dd></dl>
<p>([E] is the concentration of free enzyme).  Taken together, the general formula for the reaction rate <i>v</i> is again the Michaelis-Menten equation:</p>
<dl><dd><span class="math">
v = k_{2} [\mathrm{ES}] = \frac{k_{2} [\mbox{E}]_\mathrm{tot} [\mbox{S}]}{[\mbox{S}] + K_{m}} = \frac{V_\max [\mbox{S}]}{[\mbox{S}] + K_{m}}
</span></dd></dl>
<p>The specificity constant <span class="math">k_{cat}/K_{m}</span> is a measure of how efficiently an enzyme converts a substrate into product.  Using the definition of the Michaelis constant <span class="math">K_{m}</span>, the Michaelis-Menten equation may be written in the form </p>
<dl><dd><span class="math">
v = k_{2} [\mathrm{ES}] = \frac{k_{2}}{K_{m}} [\mbox{E}] [\mbox{S}]
</span></dd></dl>
<p>where [E] is the concentration of free enzyme.  Thus, the specificity constant is an effective <a href="/wiki/Rate_equation" id="w">second-order rate constant</a> for free enzyme to react with free substrate to form product.  The specificity constant is limited by the frequency with which the substrate and enzyme encounter each other in solution, as high as 10<sup>10</sup>&#160;<a href="/wiki/Concentration#Molarity" id="w">M</a><sup>−1</sup>&#160;<a href="/wiki/Second" id="w">s</a><sup>−1</sup>.<sup id="_ref-12" class="reference"><a href="#_note-12" title="">[12]</a></sup>  Remarkably, this maximum rate is largely unaffected by the size of either the substrate or the enzyme.<sup id="_ref-13" class="reference"><a href="#_note-13" title="">[13]</a></sup> The ratio of the specificity constants for two substrates is a quantitative comparison of how efficient the enzyme is in converting those substrates.  The slope of the Michaelis-Menten graph at low substrate concentration [S] (when [S] &#60;&#60; <i>K</i><sub>m</sub>) also yields the specificity constant.</p>
<a id="Linear_plots_of_the_Michaelis-Menten_equation" name="Linear_plots_of_the_Michaelis-Menten_equation"></a><h2>Linear plots of the Michaelis-Menten equation</h2>

<p>{{See also}}</p>
<p><div style="wigth:350px"><a class="internal" href="/wiki/Image:350px-Lineweaver-Burke_plot.svg.png" title="Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient."><img src="http://en.wikipedia.org/wiki/350px-Lineweaver-Burke_plot.svg.png" alt="Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient." title="Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient." class="location-right type-thumb" width="350px" />
</a>
<div class="thumbcaption">Lineweaver–Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient.</div></div>

Using an interactive Michaelis–Menten kinetics tutorial at the University of Virginia,{{Ref_label}} the effects on the behaviour of an enzyme of varying kinetic constants can be explored.</p>
<p>The plot of <i>v</i> versus [S] above is not linear; although initially linear at low [S], it bends over to saturate at high [S].  Before the modern era of <a href="/wiki/Nonlinear_regression" id="w">nonlinear curve-fitting</a> on computers, this nonlinearity could make it difficult to estimate <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> accurately.  Therefore, several researchers developed linearizations of the Michaelis-Menten equation, such as the <a href="/wiki/Lineweaver-Burk_plot" id="w">Lineweaver-Burk plot</a>, the <a href="/wiki/Eadie-Hofstee_diagram" id="w">Eadie-Hofstee diagram</a> and the <a href="/wiki/Hanes-Woolf_plot" id="w">Hanes-Woolf plot</a>.  All of these linear representations can be useful for visualizing data, but none should be used to determine kinetic parameters, as computer software is readily available that allows for more accurate determination by <a href="/wiki/Nonlinear_regression" id="w">nonlinear regression</a> methods.<sup id="_ref-14" class="reference"><a href="#_note-14" title="">[14]</a></sup></p>
<p>The <a href="/wiki/Lineweaver-Burk_plot" id="w">Lineweaver-Burk plot</a> or double reciprocal plot is common way of illustrating kinetic data. This is produced by taking the <a href="/wiki/Reciprocal" id="w">reciprocal</a> of both sides of the Michaelis–Menten equation. As shown on the right, this is a linear form of the Michaelis–Menten equation and produces a straight line with the equation <i>y</i> = m<i>x</i> + c with a <i>y</i>-intercept equivalent to 1/<i>V</i><sub>max</sub> and an <i>x</i>-intercept of the graph representing -1/<i>K</i><sub>m</sub>. </p>
<dl><dd><span class="math">\frac{1}{v} = \frac{K_{m}}{V_\max [\mbox{S}]} + \frac{1}{V_\max}</span></dd></dl>
<p>Naturally, no experimental values can be taken at negative 1/[S]; the lower limiting value 1/[S] = 0 (the <i>y</i>-intercept) corresponds to an infinite substrate concentration, where <i>1/v=1/V<sub>max</sub></i> as shown at the right; thus, the <i>x</i>-intercept is an <a href="/wiki/Extrapolation" id="w">extrapolation</a> of the experimental data taken at positive concentrations.  More generally, the Lineweaver-Burk plot skews the importance of measurements taken at low substrate concentrations and, thus, can yield inaccurate estimates of <i>V</i><sub>max</sub> and <i>K</i><sub>m</sub>.<sup id="_ref-15" class="reference"><a href="#_note-15" title="">[15]</a></sup>  A more accurate linear plotting method is the <a href="/wiki/Eadie-Hofstee_diagram" id="w">Eadie-Hofstee plot</a>. In this case, <i>v</i> is plotted against <i>v</i>/[S].  In the third common linear representation, the <a href="/wiki/Hanes-Woolf_plot" id="w">Hanes-Woolf plot</a>, [S]/<i>v</i> is plotted against [S].  
In general, data normalisation can help diminish the amount of experimental work and can increase the reliability of the output, and is suitable for both graphical and numerical analysis.<sup id="_ref-16" class="reference"><a href="#_note-16" title="">[16]</a></sup></p>
<a id="Practical_significance_of_kinetic_constants" name="Practical_significance_of_kinetic_constants"></a><h2>Practical significance of kinetic constants</h2>

<p>The study of enzyme kinetics is important for two basic reasons. Firstly, it helps explain how enzymes work, and secondly, it helps predict how enzymes behave in living organisms. The kinetic constants defined above, <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>, are critical to attempts to understand how enzymes work together to control <a href="/wiki/Metabolism" id="w">metabolism</a>.</p>
<p>Making these predictions is not trivial, even for simple systems. For example, <a href="/wiki/Oxaloacetate" id="w">oxaloacetate</a> is formed by <a href="/wiki/Malate_dehydrogenase" id="w">malate dehydrogenase</a> within the <a href="/wiki/Mitochondrion" id="w">mitochondrion</a>. Oxaloacetate can then be consumed by <a href="/wiki/Citrate_synthase" id="w">citrate synthase</a>, <a href="/wiki/Phosphoenolpyruvate_carboxykinase" id="w">phosphoenolpyruvate carboxykinase</a> or <a href="/wiki/Aspartate_aminotransferase" id="w">aspartate aminotransferase</a>, feeding into the <a href="/wiki/Citric_acid_cycle" id="w">citric acid cycle</a>, <a href="/wiki/Gluconeogenesis" id="w">gluconeogenesis</a> or <a href="/wiki/Aspartic_acid" id="w">aspartic acid</a> biosynthesis, respectively. Being able to predict how much oxaloacetate goes into which pathway requires knowledge of the concentration of oxaloacetate as well as the concentration and kinetics of each of these enzymes. This aim of predicting the behaviour of metabolic pathways reaches its most complex expression in the synthesis of huge amounts of kinetic and <a href="/wiki/Gene_expression" id="w">gene expression</a> data into mathematical models of entire organisms. Although this goal is far in the future for any <a href="/wiki/Eukaryote" id="w">eukaryote</a>, attempts are now being made to achieve this in <a href="/wiki/Bacteria" id="w">bacteria</a>, with models of <i>Escherichia coli</i> metabolism now being produced and tested.<sup id="_ref-17" class="reference"><a href="#_note-17" title="">[17]</a></sup><sup id="_ref-18" class="reference"><a href="#_note-18" title="">[18]</a></sup></p>
<a id="Multi-substrate_reactions" name="Multi-substrate_reactions"></a><h1> Multi-substrate reactions </h1>
<p>Multi-substrate reactions follow complex rate equations that describe how the substrates bind and in what sequence. The analysis of these reactions is much simpler if the concentration of substrate A is kept constant and substrate B varied. Under these conditions, the enzyme behaves just like a single-substrate enzyme and a plot of <i>v</i> by [S] gives apparent <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> constants for substrate B. If a set of these measurements is performed at different fixed concentrations of A, these data can be used to work out what the mechanism of the reaction is. For an enzyme that takes two substrates A and B and turns them into two products P and Q, there are two types of mechanism: ternary complex and ping–pong.</p>
<a id="Ternary-complex_mechanisms" name="Ternary-complex_mechanisms"></a><h2>Ternary-complex mechanisms</h2>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Random_order_ternary_mechanism.svg.png" title="Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line."><img src="http://en.wikipedia.org/wiki/300px-Random_order_ternary_mechanism.svg.png" alt="Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line." title="Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line.</div></div>
</p>
<p>In these enzymes, both substrates bind to the enzyme at the same time to produce an EAB ternary complex. The order of binding can either be random (in a random mechanism) or substrates have to bind in a particular sequence (in an ordered mechanism). When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ternary-complex mechanism are plotted in a <a href="/wiki/Lineweaver-Burk_plot" id="w">Lineweaver–Burk plot</a>, the set of lines produced will intersect.</p>
<p>Enzymes with ternary-complex mechanisms include <a href="/wiki/Glutathione_S-transferase" id="w">glutathione &#39;&#39;S&#39;&#39;-transferase</a>,<sup id="_ref-19" class="reference"><a href="#_note-19" title="">[19]</a></sup> <a href="/wiki/Dihydrofolate_reductase" id="w">dihydrofolate reductase</a><sup id="_ref-20" class="reference"><a href="#_note-20" title="">[20]</a></sup> and <a href="/wiki/DNA_polymerase" id="w">DNA polymerase</a>.<sup id="_ref-21" class="reference"><a href="#_note-21" title="">[21]</a></sup> The following links show short animations of the ternary-complex mechanisms of the enzymes dihydrofolate reductase{{Ref_label}} and DNA polymerase{{Ref_label}}.</p>
<a id="Ping%E2%80%93pong_mechanisms" name="Ping%E2%80%93pong_mechanisms"></a><h2>Ping–pong mechanisms</h2>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Ping_pong.svg.png" title="Ping–pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q."><img src="http://en.wikipedia.org/wiki/300px-Ping_pong.svg.png" alt="Ping–pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q." title="Ping–pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Ping–pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q.</div></div>

As shown on the right, enzymes with a ping-pong mechanism can exist in two states, E and a chemically modified form of the enzyme E*; this modified enzyme is known as an <a href="/wiki/Reactive_intermediate" id="w">intermediate</a>. In such mechanisms, substrate A binds, changes the enzyme to E* by, for example, transferring a chemical group to the active site, and is then released. Only after the first substrate is released can substrate B bind and react with the modified enzyme, regenerating the unmodified E form. When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ping–pong mechanism are plotted in a Lineweaver–Burk plot, a set of parallel lines will be produced.</p>
<p>Enzymes with ping–pong mechanisms include some <a href="/wiki/Oxidoreductases" id="w">oxidoreductases</a> such as <a href="/wiki/Peroxidase" id="w">thioredoxin peroxidase</a>,<sup id="_ref-22" class="reference"><a href="#_note-22" title="">[22]</a></sup> <a href="/wiki/Transferases" id="w">transferases</a> such as acylneuraminate cytydilyltransferase<sup id="_ref-23" class="reference"><a href="#_note-23" title="">[23]</a></sup> and <a href="/wiki/Serine_protease" id="w">serine proteases</a> such as <a href="/wiki/Trypsin" id="w">trypsin</a> and <a href="/wiki/Chymotrypsin" id="w">chymotrypsin</a>.<sup id="_ref-24" class="reference"><a href="#_note-24" title="">[24]</a></sup>  Serine proteases are a very common and diverse family of enzymes, including <a href="/wiki/Digestion" id="w">digestive</a> enzymes (trypsin, chymotrypsin, and elastase), several enzymes of the <a href="/wiki/Coagulation" id="w">blood clotting cascade</a> and many others. In these serine proteases, the E* intermediate is an acyl-enzyme species formed by the attack of an active site <a href="/wiki/Serine" id="w">serine</a> residue on a <a href="/wiki/Peptide_bond" id="w">peptide bond</a> in a protein substrate. A short animation showing the mechanism of chymotrypsin is linked here.{{Ref_label}}</p>
<a id="Non-Michaelis%E2%80%93Menten_kinetics" name="Non-Michaelis%E2%80%93Menten_kinetics"></a><h1> Non-Michaelis–Menten kinetics </h1>
<p>{{main}}
<div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Allosteric_v_by_S_curve.svg.png" title="Saturation curve for an enzyme reaction showing sigmoid kinetics."><img src="http://en.wikipedia.org/wiki/300px-Allosteric_v_by_S_curve.svg.png" alt="Saturation curve for an enzyme reaction showing sigmoid kinetics." title="Saturation curve for an enzyme reaction showing sigmoid kinetics." class="location-left type-thumb" width="300px" />
</a>
<div class="thumbcaption">Saturation curve for an enzyme reaction showing sigmoid kinetics.</div></div>

Some enzymes produce a <a href="/wiki/Sigmoid_function" id="w">sigmoid</a> <i>v</i> by [S] plot, which often indicates <a href="/wiki/Cooperative_binding" id="w">cooperative binding</a> of substrate to the active site. This means that the binding of one substrate molecule affects the binding of subsequent substrate molecules. This behaviour is most common in <a href="/wiki/Protein_structure" id="w">multimeric</a> enzymes with several interacting active sites.<sup id="_ref-25" class="reference"><a href="#_note-25" title="">[25]</a></sup> Here, the mechanism of cooperation is similar to that of <a href="/wiki/Hemoglobin" id="w">haemoglobin</a>, with binding of substrate to one active site altering the affinity of the other active sites for substrate molecules. Positive cooperativity occurs when binding of the first substrate molecule <i>increases</i> the affinity of the other active sites for substrate. Negative cooperativity occurs when binding of the first substrate <i>decreases</i> the affinity of the enzyme for other substrate molecules.</p>
<p>Allosteric enzymes include mammalian tyrosyl tRNA-synthetase, which shows negative cooperativity,<sup id="_ref-26" class="reference"><a href="#_note-26" title="">[26]</a></sup> and bacterial <a href="/wiki/Aspartate_transcarbamoylase" id="w">aspartate transcarbamoylase</a><sup id="_ref-27" class="reference"><a href="#_note-27" title="">[27]</a></sup> and <a href="/wiki/Phosphofructokinase" id="w">phosphofructokinase</a><sup id="_ref-28" class="reference"><a href="#_note-28" title="">[28]</a></sup>, which show positive cooperativity.</p>
<p>Cooperativity is surprisingly common and can help regulate the responses of enzymes to changes in the concentrations of their substrates. Positive cooperativity makes enzymes much more sensitive to [S] and their activities can show large changes over a narrow range of substrate concentration. Conversely, negative cooperativity makes enzymes insensitive to small changes in [S].</p>
<p>The <a href="/wiki/Hill_equation" id="w">Hill equation</a><sup id="_ref-29" class="reference"><a href="#_note-29" title="">[29]</a></sup> is often used to describe the degree of cooperativity quantitatively in non-Michaelis–Menten kinetics.  The derived Hill coefficient <i>n</i> measures how much the binding of substrate to one active site affects the binding of substrate to the other active sites. A Hill coefficient of &#60;1 indicates negative cooperativity and a coefficient of &#62;1 indicates positive <a href="/wiki/Cooperativity" id="w">cooperativity</a>.</p>
<a id="Pre-steady-state_kinetics" name="Pre-steady-state_kinetics"></a><h1> Pre-steady-state kinetics </h1>
<p><div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Burst_phase.png" title="Pre-steady state progress curve, showing the burst phase of an enzyme reaction."><img src="http://en.wikipedia.org/wiki/300px-Burst_phase.png" alt="Pre-steady state progress curve, showing the burst phase of an enzyme reaction." title="Pre-steady state progress curve, showing the burst phase of an enzyme reaction." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Pre-steady state progress curve, showing the burst phase of an enzyme reaction.</div></div>

In the first moment after an enzyme is mixed with substrate, no product has been formed and no <a href="/wiki/Reactive_intermediate" id="w">intermediates</a> exist. The study of the next few milliseconds of the reaction is called pre-steady-state kinetics. Pre-steady-state kinetics is therefore concerned with the formation and consumption of enzyme–substrate intermediates (such as ES or E*) until their <a href="/wiki/Steady_state_(chemistry)" id="w">steady-state concentrations</a> are reached.</p>
<p>This approach was first applied to the hydrolysis reaction catalysed by <a href="/wiki/Chymotrypsin" id="w">chymotrypsin</a>.<sup id="_ref-30" class="reference"><a href="#_note-30" title="">[30]</a></sup> Often, the detection of an intermediate is a vital piece of evidence in investigations of what mechanism an enzyme follows. For example, in the ping–pong mechanisms that are shown above, rapid kinetic measurements can follow the release of product P and measure the formation of the modified enzyme intermediate E*.<sup id="_ref-Fersht_a" class="reference"><a href="#_note-Fersht" title="">[31]</a></sup> In the case of chymotrypsin, this intermediate is formed by the attack of the substrate by the <a href="/wiki/Nucleophile" id="w">nucleophilic</a> serine in the active site and the formation of the acyl-enzyme intermediate.</p>
<p>In the figure to the right, the enzyme produces E* rapidly in the first few seconds of the reaction. The rate then slows as steady state is reached. This rapid burst phase of the reaction measures a single turnover of the enzyme. Consequently, the amount of product released in this burst, shown as the intercept on the <i>y</i>-axis of the graph, also gives the amount of functional enzyme which is present in the assay.<sup id="_ref-32" class="reference"><a href="#_note-32" title="">[32]</a></sup></p>
<a id="Chemical_mechanism" name="Chemical_mechanism"></a><h1>Chemical mechanism</h1>

<p>An important goal of measuring enzyme kinetics is to determine the chemical mechanism of an enzyme reaction, i.e., the sequence of chemical steps that transform substrate into product. The kinetic approaches discussed above will show at what rates <a href="/wiki/Reactive_intermediate" id="w">intermediates</a> are formed and inter-converted, but they cannot identify exactly what these intermediates are.</p>
<p>Kinetic measurements taken under various solution conditions or on slightly modified enzymes or substrates often shed light on this chemical mechanism, as they reveal the rate-determining step or intermediates in the reaction. For example, the breaking of a <a href="/wiki/Covalent_bond" id="w">covalent bond</a> to a <a href="/wiki/Hydrogen" id="w">hydrogen</a> <a href="/wiki/Atom" id="w">atom</a> is a common rate-determining step. Which of the possible hydrogen transfers is rate determining can be shown by measuring the kinetic effects of substituting each hydrogen by <a href="/wiki/Deuterium" id="w">deuterium</a>, its stable <a href="/wiki/Isotope" id="w">isotope</a>. The rate will change when the critical hydrogen is replaced, due to a primary <a href="/wiki/Kinetic_isotope_effect" id="w">kinetic isotope effect</a>, which occurs because bonds to deuterium are harder to break then bonds to hydrogen.<sup id="_ref-33" class="reference"><a href="#_note-33" title="">[33]</a></sup> It is also possible to measure similar effects with other isotope substitutions, such as <sup>13</sup>C/<sup>12</sup>C and <sup>18</sup>O/<sup>16</sup>O, but these effects are more subtle.<sup id="_ref-34" class="reference"><a href="#_note-34" title="">[34]</a></sup></p>
<p>Isotopes can also be used to reveal the fate of various parts of the substrate molecules in the final products. For example, it is sometimes difficult to discern the origin of an <a href="/wiki/Oxygen" id="w">oxygen</a> atom in the final product; since it may have come from water or from part of the substrate. This may be determined by systematically substituting oxygen&#39;s stable isotope <sup>18</sup>O into the various molecules that participate in the reaction and checking for the isotope in the product.<sup id="_ref-35" class="reference"><a href="#_note-35" title="">[35]</a></sup> The chemical mechanism can also be elucidated by examining the kinetics and isotope effects under different pH conditions,<sup id="_ref-36" class="reference"><a href="#_note-36" title="">[36]</a></sup> by altering the metal ions or other bound <a href="/wiki/Cofactor_(biochemistry)" id="w">cofactors</a>,<sup id="_ref-37" class="reference"><a href="#_note-37" title="">[37]</a></sup>  by <a href="/wiki/Site-directed_mutagenesis" id="w">site-directed mutagenesis</a> of conserved amino acid residues, or by studying the behaviour of the enzyme in the presence of analogues of the substrate(s).<sup id="_ref-38" class="reference"><a href="#_note-38" title="">[38]</a></sup></p>
<a id="Enzyme_inhibition" name="Enzyme_inhibition"></a><h1>Enzyme inhibition</h1>
<p>{{main}}
<div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Reversible_inhibition.svg.png" title="Kinetic scheme for reversible enzyme inhibitors."><img src="http://en.wikipedia.org/wiki/300px-Reversible_inhibition.svg.png" alt="Kinetic scheme for reversible enzyme inhibitors." title="Kinetic scheme for reversible enzyme inhibitors." class="location-right type-thumb" width="300px" />
</a>
<div class="thumbcaption">Kinetic scheme for reversible enzyme inhibitors.</div></div>
</p>
<p>Enzyme inhibitors are molecules that reduce or abolish enzyme activity. These are either <i>reversible</i> (i.e., removal of the inhibitor restores enzyme activity) or <i>irreversible</i> (i.e., the inhibitor permanently inactivates the enzyme).</p>
<a id="Reversible_inhibitors" name="Reversible_inhibitors"></a><h2>Reversible inhibitors</h2>

<p>Reversible enzyme inhibitors can be classified as competitive, uncompetitive, non-competitive or mixed, according to their effects on <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>. These different effects result from the inhibitor binding to the enzyme E, to the enzyme–substrate complex ES, or to both, as shown in the figure to the right and the table below.  The particular type of an inhibitor can be discerned by studying the enzyme kinetics as a function of the inhibitor concentration.  The four types of inhibition produce <a href="/wiki/Lineweaver-Burk_plot" id="w">Lineweaver–Burke</a> and <a href="/wiki/Eadie-Hofstee_diagram" id="w">Eadie–Hofstee</a> plots<sup id="_ref-39" class="reference"><a href="#_note-39" title="">[39]</a></sup> that vary in distinctive ways with inhibitor concentration.  For brevity, two symbols are used:</p>
<dl><dd><span class="math"> \alpha = 1 + \frac{[\mbox{I}]}{K_{i}} </span>&#160;&#160;&#160;&#160;&#160;&#160;and&#160;&#160;&#160;&#160;&#160;&#160;<span class="math">\alpha^{\prime} = 1 + \frac{[\mbox{I}]}{K_{i}^{\prime}}</span></dd></dl>
<p>where <i>K</i><sub>i</sub> and <i>K<b><sub>i</sub> are the <a href="/wiki/Dissociation_constant" id="w">dissociation constants</a> for binding to the enzyme and to the enzyme–substrate complex, respectively.  In the presence of the reversible inhibitor, the enzyme&#39;s apparent <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> become (α/α&#39;)<i>K</i><sub>m</sub> and (1/α&#39;)<i>V</i><sub>max</sub>, respectively, as shown below for common cases.</b></i></p>

<div style="page-break-inside:	avoid;">
<table align="center" border="0" width="75%">
<tr>
<td align="center" width="10%" />
<td align="center" width="10%" />
<td align="center" bgcolor="#FFC3C3" width="20%"><b>Type of inhibition</b></td>
<td align="center" bgcolor="#FFC3C3" width="20%"><b><i>K</i></b><b><sub>m</sub> apparent</b></td>
<td align="center" bgcolor="#FFC3C3" width="20%"><b><i>V</i></b><b><sub>max</sub> apparent</b>
</td></tr>
<tr>
<td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> only</td>
<td align="center">(<span class="math">\alpha^{\prime}=1</span>)</td>
<td align="center" bgcolor="#DDFFDD">competitive</td>
<td align="center"><span class="math">K_m \alpha~</span></td>
<td align="center"><span class="math">~V_\max ~</span>
</td></tr>
<tr>
<td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub>&#39; only</td>
<td align="center">(<span class="math">\alpha=1~</span>) </td>
<td align="center" bgcolor="#DDFFDD">uncompetitive</td>
<td align="center"><span class="math">\frac{K_m}{\alpha^{\prime}}</span></td>
<td align="center"><span class="math">\frac{V_\max}{\alpha^{\prime}}</span>
</td></tr>
<tr>
<td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> = <i>K</i><sub>i</sub>&#39;</td>
<td align="center">(<span class="math">\alpha = \alpha^{\prime}</span>) </td>
<td align="center" bgcolor="#DDFFDD">non-competitive</td>
<td align="center"><span class="math">~K_m~</span></td>
<td align="center"><span class="math">\frac{V_\max}{\alpha^{\prime}}</span>
</td></tr>
<tr>
<td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> ≠ <i>K</i><sub>i</sub>&#39;  </td>
<td align="center">(<span class="math">\alpha \neq \alpha^{\prime}</span>)</td>
<td align="center" bgcolor="#DDFFDD">mixed</td>
<td align="center"><span class="math">\frac{K_m\alpha}{\alpha^{\prime}}</span></td>
<td align="center"><span class="math">\frac{V_\max}{\alpha^{\prime}}</span></td></tr></table></div>

<p><a href="/wiki/Non-linear_regression" id="w">Non-linear regression</a> fits of the enzyme kinetics data to the rate equations above<sup id="_ref-40" class="reference"><a href="#_note-40" title="">[40]</a></sup> can yield accurate estimates of the dissociation constants <i>K</i><sub>i</sub> and <i>K<b><sub>i</sub>.</b></i></p>
<a id="Irreversible_inhibitors" name="Irreversible_inhibitors"></a><h2>Irreversible inhibitors</h2>

<p>Enzyme inhibitors can also irreversibly inactivate enzymes, usually by covalently modifying active site residues. These reactions follow <a href="/wiki/Exponential_decay" id="w">exponential decay</a> functions and are usually saturable. Below saturation, they follow <a href="/wiki/Reaction_rate" id="w">first order</a> kinetics with respect to inhibitor.</p>
<a id="Mechanisms_of_catalysis" name="Mechanisms_of_catalysis"></a><h1>Mechanisms of catalysis</h1>
<p>{{main}}
<div style="wigth:300px"><a class="internal" href="/wiki/Image:300px-Activation2_updated.svg.png" title="The energy variation as a function of [[reaction coordinate]] shows the stabilisation of the transition state by an enzyme."><img src="http://en.wikipedia.org/wiki/300px-Activation2_updated.svg.png" alt="The energy variation as a function of [[reaction coordinate]] shows the stabilisation of the transition state by an enzyme." title="The energy variation as a function of [[reaction coordinate]] shows the stabilisation of the transition state by an enzyme." class="location-none type-thumb" width="300px" />
</a>
<div class="thumbcaption">The energy variation as a function of <a href="/wiki/Reaction_coordinate" id="w">reaction coordinate</a> shows the stabilisation of the transition state by an enzyme.</div></div>
</p>
<p>The favoured model for the enzyme–substrate interaction is the induced fit model.<sup id="_ref-41" class="reference"><a href="#_note-41" title="">[41]</a></sup> This model proposes that the initial interaction between enzyme and substrate is relatively weak, but that these weak interactions rapidly induce conformational changes in the enzyme that strengthen binding. These <a href="/wiki/Tertiary_structure" id="w">conformational</a> changes also bring catalytic residues in the active site close to the chemical bonds in the substrate that will be altered in the reaction.<sup id="_ref-42" class="reference"><a href="#_note-42" title="">[42]</a></sup> After binding takes place, one or more mechanisms of catalysis lower the energy of the reaction&#39;s <a href="/wiki/Transition_state" id="w">transition state</a> by providing an alternative chemical pathway for the reaction. Mechanisms of catalysis include catalysis by bond strain; by proximity and orientation; by active-site proton donors or acceptors; covalent catalysis and quantum tunnelling.<sup id="_ref-Fersht_b" class="reference"><a href="#_note-Fersht" title="">[31]</a></sup><sup id="_ref-43" class="reference"><a href="#_note-43" title="">[43]</a></sup></p>
<p>Enzyme kinetics cannot prove which modes of catalysis are used by an enzyme. However, some kinetic data can suggest possibilities to be examined by other techniques. For example, a ping–pong mechanism with burst-phase pre-steady-state kinetics would suggest covalent catalysis might be important in this enzyme&#39;s mechanism. Alternatively, the observation of a strong pH effect on <i>V</i><sub>max</sub> but not <i>K</i><sub>m</sub> might indicate that a residue in the active site needs to be in a particular <a href="/wiki/Ionization" id="w">ionisation</a> state for catalysis to occur.</p>
<a id="Footnotes" name="Footnotes"></a><h1>Footnotes</h1>

<p><b>α.</b> {{Note_label}}<a class="externallink" href="http://cti.itc.virginia.edu/~cmg/Demo/scriptFrame.html" rel="nofollow" title="http://cti.itc.virginia.edu/~cmg/Demo/scriptFrame.html">Link: Interactive Michaelis–Menten kinetics tutorial (Java required)</a></p>
<p><b>β.</b> {{Note_label}}<a class="externallink" href="http://chem-faculty.ucsd.edu/kraut/dhfr.html" rel="nofollow" title="http://chem-faculty.ucsd.edu/kraut/dhfr.html">Link: dihydrofolate reductase mechanism (Gif)</a></p>
<p><b>γ.</b> {{Note_label}}<a class="externallink" href="http://chem-faculty.ucsd.edu/kraut/dNTP.html" rel="nofollow" title="http://chem-faculty.ucsd.edu/kraut/dNTP.html">Link: DNA polymerase mechanism (Gif)</a></p>
<p><b>δ.</b> {{Note_label}}<a class="externallink" href="http://courses.cm.utexas.edu/jrobertus/ch339k/overheads-2/06_21_chymotrypsin.html" rel="nofollow" title="http://courses.cm.utexas.edu/jrobertus/ch339k/overheads-2/06_21_chymotrypsin.html">Link: Chymotrypsin mechanism (Flash required)</a></p>
<a id="References" name="References"></a><h1> References </h1>
<p />
<div class="references-2column">
<p /><ol class="references">
<li id="_note-1"><b><a href="#_ref-1" title="">&uarr;</a></b> Ebbing, D.D. <i>General chemistry</i> (4th edition) Houghton Mifflin Co. 1993, <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0395636965" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0395636965">ISBN 0-395-63696-5</a></li><li id="_note-2"><b><a href="#_ref-2" title="">&uarr;</a></b> Eisenthal R. Danson M.J. (Eds), <i>Enzyme Assays: A Practical Approach.</i> Oxford University Press (2002) <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0199638209" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0199638209">ISBN 0-19-963820-9</a></li><li id="_note-3"><b><a href="#_ref-3" title="">&uarr;</a></b> Xie XS, Lu HP. <a class="externallink" href="http://www.jbc.org/cgi/content/full/274/23/15967" rel="nofollow" title="http://www.jbc.org/cgi/content/full/274/23/15967">&#39;&#39;Single-molecule enzymology.&#39;&#39;</a> J Biol Chem. 1999 Jun 4;274(23):15967-70. PMID 10347141</li><li id="_note-4"><b><a href="#_ref-4" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-5"><b><a href="#_ref-5" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-6"><b><a href="#_ref-6" title="">&uarr;</a></b> Gibson Q.H. Rapid mixing: Stopped flow <i>Methods in Enzymology</i>, (1969) 16:187–228</li><li id="_note-7"><b><a href="#_ref-7" title="">&uarr;</a></b> Duggleby, R.G. Analysis of enzyme progress curves by non-linear regression. <i>Methods in Enzymology</i>, (1995) 249:61–90.</li><li id="_note-8"><b><a href="#_ref-8" title="">&uarr;</a></b> Hammann C, Lilley DM. <i>Folding and activity of the hammerhead ribozyme.</i> Chembiochem. 2002 Aug 2;3(8):690–700. PMID 11779233</li><li id="_note-9"><b><a href="#_ref-9" title="">&uarr;</a></b> Michaelis L. and Menten M.L. <i>Kinetik der Invertinwirkung</i> Biochem. Z. 1913; 49:333–369 <a class="externallink" href="http://web.lemoyne.edu/~giunta/menten.html" rel="nofollow" title="http://web.lemoyne.edu/~giunta/menten.html">English translation</a> Accessed 6 April 2007</li><li id="_note-Cha">&uarr; <a href="#_ref-Cha_a" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Cha_b" title=""><sup><i><b>b</b></i></sup></a> {{cite journal}}</li><li id="_note-11"><b><a href="#_ref-11" title="">&uarr;</a></b> Briggs GE, Haldane JB. <a class="externallink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1259181&#38;blobtype=pdf" rel="nofollow" title="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1259181&#38;blobtype=pdf">&#39;&#39;A Note on the Kinetics of Enzyme Action.&#39;&#39;</a> Biochem J. 1925;19(2):338-9. PMID 16743508</li><li id="_note-12"><b><a href="#_ref-12" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-13"><b><a href="#_ref-13" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-14"><b><a href="#_ref-14" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-15"><b><a href="#_ref-15" title="">&uarr;</a></b> Tseng SJ, Hsu JP. <i>A comparison of the parameter estimating procedures for the Michaelis–Menten model.</i> J Theor Biol. 1990 Aug 23;145(4):457–64. PMID 2246896</li><li id="_note-16"><b><a href="#_ref-16" title="">&uarr;</a></b> Bravo, I.G., Busto, F., De Arriaga, D., Ferrero, M. A., Rodríguez-Aparicio, L. B., Martínez-Blanco H., Reglero, A.
<a class="externallink" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&#38;pubmedid=11577687" rel="nofollow" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&#38;pubmedid=11577687">&#39;&#39;A normalised plot as a novel and time-saving tool in complex enzyme kinetic analysis&#39;&#39;</a> Biochem. J. (2001). 358, 573–583.
PMID 11577687</li><li id="_note-17"><b><a href="#_ref-17" title="">&uarr;</a></b> Almaas E, Kovacs B, Vicsek T, Oltvai ZN, Barabasi AL. <i>Global organization of metabolic fluxes in the bacterium Escherichia coli.</i> Nature. 2004 Feb 26;427(6977):839-43. PMID 14985762</li><li id="_note-18"><b><a href="#_ref-18" title="">&uarr;</a></b> Reed JL, Vo TD, Schilling CH, Palsson BO. <a class="externallink" href="http://genomebiology.com/2003/4/9/R54" rel="nofollow" title="http://genomebiology.com/2003/4/9/R54">&#39;&#39;An expanded genome-scale model of Escherichia coli K-12 (iJR904 GSM/GPR).&#39;&#39;</a> Genome Biol. 2003;4(9):R54. PMID 12952533</li><li id="_note-19"><b><a href="#_ref-19" title="">&uarr;</a></b> Dirr H, Reinemer P, Huber R. <i>X-ray crystal structures of cytosolic glutathione S-transferases. Implications for protein architecture, substrate recognition and catalytic function.</i> Eur J Biochem. 1994 Mar 15;220(3):645-61. PMID 8143720</li><li id="_note-20"><b><a href="#_ref-20" title="">&uarr;</a></b> Stone SR, Morrison JF. <i>Dihydrofolate reductase from Escherichia coli: the kinetic mechanism with NADPH and reduced acetylpyridine adenine dinucleotide phosphate as substrates.</i> Biochemistry. 1988 Jul 26;27(15):5493–9. PMID 3052577</li><li id="_note-21"><b><a href="#_ref-21" title="">&uarr;</a></b> Fisher PA. <i>Enzymologic mechanism of replicative DNA polymerases in higher eukaryotes.</i> Prog Nucleic Acid Res Mol Biol. 1994;47:371-97. PMID 8016325</li><li id="_note-22"><b><a href="#_ref-22" title="">&uarr;</a></b> Akerman SE, Muller S. <a class="externallink" href="http://www.sciencedirect.com/science?_ob=ArticleURL&#38;_udi=B6T29-49321S6-2&#38;_coverDate=08%2F31%2F2003&#38;_alid=466052639&#38;_rdoc=1&#38;_fmt=&#38;_orig=search&#38;_qd=1&#38;_cdi=4913&#38;_sort=d&#38;view=c&#38;_acct=C000050221&#38;_version=1&#38;_urlVersion=0&#38;_userid=10&#38;md5=965e18a4c2ad0af711ba05d1a46dc855" rel="nofollow" title="http://www.sciencedirect.com/science?_ob=ArticleURL&#38;_udi=B6T29-49321S6-2&#38;_coverDate=08%2F31%2F2003&#38;_alid=466052639&#38;_rdoc=1&#38;_fmt=&#38;_orig=search&#38;_qd=1&#38;_cdi=4913&#38;_sort=d&#38;view=c&#38;_acct=C000050221&#38;_version=1&#38;_urlVersion=0&#38;_userid=10&#38;md5=965e18a4c2ad0af711ba05d1a46dc855">&#39;&#39;2-Cys peroxiredoxin PfTrx-Px1 is involved in the antioxidant defence of Plasmodium falciparum.&#39;&#39;</a> Mol Biochem Parasitol. 2003 Aug 31;130(2):75-81. PMID 12946843</li><li id="_note-23"><b><a href="#_ref-23" title="">&uarr;</a></b> Bravo, I.G., Barrallo, S., Ferrero, M. A., Rodríguez-Aparicio, L. B., Martínez-Blanco H., Reglero, A.
“Kinetic properties of the Acylneuraminate Cytidylytransferase from Pasteurella haemolytica A2”. Biochem. J. (2001) 358, 585-598.
<a class="externallink" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&#38;pubmedid=11577688" rel="nofollow" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&#38;pubmedid=11577688">http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&#38;pubmedid=11577688</a></li><li id="_note-24"><b><a href="#_ref-24" title="">&uarr;</a></b> Kraut J. <i>Serine proteases: structure and mechanism of catalysis.</i> Annu Rev Biochem. 1977;46:331-58. PMID 332063</li><li id="_note-25"><b><a href="#_ref-25" title="">&uarr;</a></b> Ricard J, Cornish-Bowden A. <i>Co-operative and allosteric enzymes: 20 years on.</i> Eur J Biochem. 1987 Jul 15;166(2):255-72. PMID 3301336</li><li id="_note-26"><b><a href="#_ref-26" title="">&uarr;</a></b> Ward WH, Fersht AR., <i>Tyrosyl-tRNA synthetase acts as an asymmetric dimer in charging tRNA. A rationale for half-of-the-sites activity.</i> Biochemistry. 1988 Jul 26;27(15):5525–30. PMID 3179266</li><li id="_note-27"><b><a href="#_ref-27" title="">&uarr;</a></b> Helmstaedt K, Krappmann S, Braus GH., <a class="externallink" href="http://mmbr.asm.org/cgi/content/full/65/3/404?view=long&#38;pmid=11528003" rel="nofollow" title="http://mmbr.asm.org/cgi/content/full/65/3/404?view=long&#38;pmid=11528003">&#39;&#39;Allosteric regulation of catalytic activity: Escherichia coli aspartate transcarbamoylase versus yeast chorismate mutase.&#39;&#39;</a>  Microbiol. Mol. Biol. Rev. 2001 Sep;65(3):404-21 PMID 11528003</li><li id="_note-28"><b><a href="#_ref-28" title="">&uarr;</a></b> Schirmer T, Evans PR., <i>Structural basis of the allosteric behaviour of phosphofructokinase.</i> Nature. 1990 Jan 11;343(6254):140-5. PMID 2136935</li><li id="_note-29"><b><a href="#_ref-29" title="">&uarr;</a></b> Hill, A. V.  The possible effects of the aggregation of the molecules of haemoglobin on its dissociation curves. <i>J. Physiol. (Lond.)</i>, 1910 40, iv-vii.</li><li id="_note-30"><b><a href="#_ref-30" title="">&uarr;</a></b> Hartley B.S. and Kilby B.A. <a class="externallink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1269615&#38;blobtype=pdf" rel="nofollow" title="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1269615&#38;blobtype=pdf">&#39;&#39;The reaction of p-nitrophenyl esters with chymotrypsin and insulin.&#39;&#39;</a> Biochem J. 1954 Feb;56(2):288-97. PMID 13140189</li><li id="_note-Fersht">&uarr; <a href="#_ref-Fersht_a" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Fersht_b" title=""><sup><i><b>b</b></i></sup></a> Alan Fersht, <i>Structure and Mechanism in Protein Science : A Guide to Enzyme Catalysis and Protein Folding.</i> W. H. Freeman, 1998. <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0716732688" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0716732688">ISBN 0-7167-3268-8</a></li><li id="_note-32"><b><a href="#_ref-32" title="">&uarr;</a></b> Bender ML, Begue-Canton ML, Blakeley RL, Brubacher LJ, Feder J, Gunter CR, Kezdy FJ, Killheffer JV Jr, Marshall TH, Miller CG, Roeske RW, Stoops JK. <i>The Determination of the Concentration of Hydrolytic Enzyme Solutions : a-Chymotrypsin, Trypsin, Papain, Elastase, Subtilisin, and Acetylcholinesterase.</i> J Am Chem Soc. 1966 Dec 20;88(24):5890-913. PMID 5980876</li><li id="_note-33"><b><a href="#_ref-33" title="">&uarr;</a></b> Cleland WW. <a class="externallink" href="http://www.sciencedirect.com/science?_ob=ArticleURL&#38;_udi=B6WB5-4DD8DXJ-8&#38;_coverDate=01%2F01%2F2005&#38;_alid=466049795&#38;_rdoc=1&#38;_fmt=&#38;_orig=search&#38;_qd=1&#38;_cdi=6701&#38;_sort=d&#38;view=c&#38;_acct=C000050221&#38;_version=1&#38;_urlVersion=0&#38;_userid=10&#38;md5=cf322c4a1c7db6b9f89551a8469a1a2d" rel="nofollow" title="http://www.sciencedirect.com/science?_ob=ArticleURL&#38;_udi=B6WB5-4DD8DXJ-8&#38;_coverDate=01%2F01%2F2005&#38;_alid=466049795&#38;_rdoc=1&#38;_fmt=&#38;_orig=search&#38;_qd=1&#38;_cdi=6701&#38;_sort=d&#38;view=c&#38;_acct=C000050221&#38;_version=1&#38;_urlVersion=0&#38;_userid=10&#38;md5=cf322c4a1c7db6b9f89551a8469a1a2d">&#39;&#39;The use of isotope effects to determine enzyme mechanisms.&#39;&#39;</a> Arch Biochem Biophys. 2005 Jan 1;433(1):2–12. PMID 15581561</li><li id="_note-34"><b><a href="#_ref-34" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-35"><b><a href="#_ref-35" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-36"><b><a href="#_ref-36" title="">&uarr;</a></b> Cleland WW. <i>Use of isotope effects to elucidate enzyme mechanisms.</i> CRC Crit Rev Biochem. 1982;13(4):385–428. PMID 6759038</li><li id="_note-37"><b><a href="#_ref-37" title="">&uarr;</a></b> Christianson DW, Cox JD. <i>Catalysis by metal-activated hydroxide in zinc and manganese metalloenzymes.</i> Annu Rev Biochem. 1999;68:33–57. PMID 10872443</li><li id="_note-38"><b><a href="#_ref-38" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-39"><b><a href="#_ref-39" title="">&uarr;</a></b> Tseng SJ, Hsu JP. <i>A comparison of the parameter estimating procedures for the Michaelis–Menten model.</i> J Theor Biol. 1990 Aug 23;145(4):457–64. PMID 2246896</li><li id="_note-40"><b><a href="#_ref-40" title="">&uarr;</a></b> Leatherbarrow RJ. <i>Using linear and non-linear regression to fit biochemical data.</i> Trends Biochem Sci. 1990 Dec;15(12):455–8. PMID 2077683</li><li id="_note-41"><b><a href="#_ref-41" title="">&uarr;</a></b> Koshland DE, <a class="externallink" href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=335371&#38;blobtype=pdf" rel="nofollow" title="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=335371&#38;blobtype=pdf">&#39;&#39;Application of a Theory of Enzyme Specificity to Protein Synthesis.&#39;&#39;</a> Proc. Natl. Acad. Sci. U.S.A. 1958 Feb;44(2):98–104. PMID 16590179</li><li id="_note-42"><b><a href="#_ref-42" title="">&uarr;</a></b> {{cite journal}}</li><li id="_note-43"><b><a href="#_ref-43" title="">&uarr;</a></b> {{cite journal}}</li></ol></div>

<a id="Further_reading" name="Further_reading"></a><h1>Further reading</h1>


<ul>
<li>Athel Cornish-Bowden, <i>Fundamentals of Enzyme Kinetics</i>. (3rd edition), Portland Press 2004, <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/1855781581" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/1855781581">ISBN 1-85578-158-1</a>.</li>
<li>Irwin H. Segel, <i>Enzyme Kinetics : Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</i>. Wiley-Interscience; New Ed edition 1993, <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0471303097" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0471303097">ISBN 0-471-30309-7</a>.</li>
<li>Alan Fersht, <i>Structure and Mechanism in Protein Science : A Guide to Enzyme Catalysis and Protein Folding.</i> W. H. Freeman, 1998. <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0716732688" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0716732688">ISBN 0-7167-3268-8</a></li>
<li>Santiago Schnell, Philip K. Maini, A century of enzyme kinetics: Reliability of the K<sub>M</sub> and v<sub>max</sub> estimates, <i>Comments on Theoretical Biology</i> <b>8</b>, 169–187, 2004 DOI: <a class="externallink" href="http://www.informatics.indiana.edu/schnell/papers/ctb8_169.pdf" rel="nofollow" title="http://www.informatics.indiana.edu/schnell/papers/ctb8_169.pdf">10.1080/08948550390206768</a></li>
<li>Chris Walsh, <i>Enzymatic Reaction Mechanisms</i>. W. H. Freeman and Company. 1979. <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/0716700700" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/0716700700">ISBN 0-7167-0070-0</a> </li>
<li>Nicholas Price, Lewis Stevens, <i>Fundamentals of Enzymology</i>, Oxford University Press, 1999. <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/019850229" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/019850229">ISBN 0-19-850229-</a>X</li>
<li>Tim Bugg, <i>An Introduction to Enzyme and Coenzyme Chemistry</i> Blackwell Publishing, 2004 <a class="external text" href="http://www.amazon.com/exec/obidos/ASIN/1405114525" rel="nofollow" title="http://www.amazon.com/exec/obidos/ASIN/1405114525">ISBN 1-4051-1452-5</a></li></ul>
<a id="External_links" name="External_links"></a><h1>External links</h1>


<ul>
<li><a class="externallink" href="http://www.kscience.co.uk/animations/model.swf" rel="nofollow" title="http://www.kscience.co.uk/animations/model.swf">Animation of an enzyme assay</a> — Shows effects of manipulating assay conditions</li>
<li><a class="externallink" href="http://www.ebi.ac.uk/thornton-srv/databases/MACiE/" rel="nofollow" title="http://www.ebi.ac.uk/thornton-srv/databases/MACiE/">MACiE</a> — A database of enzyme reaction mechanisms</li>
<li><a class="externallink" href="http://us.expasy.org/enzyme/" rel="nofollow" title="http://us.expasy.org/enzyme/">ENZYME</a> — Expasy enzyme nomenclature database</li>
<li><a class="externallink" href="http://mbs.cbrc.jp/EzCatDB/" rel="nofollow" title="http://mbs.cbrc.jp/EzCatDB/">ExCatDB</a> — A database of enzyme catalytic mechanisms</li>
<li><a class="externallink" href="http://www.brenda-enzymes.info/" rel="nofollow" title="http://www.brenda-enzymes.info/">BRENDA</a> — Comprehensive enzyme database, giving substrates, inhibitors and reaction diagrams</li>
<li><a class="externallink" href="http://sabio.villa-bosch.de/SABIORK/" rel="nofollow" title="http://sabio.villa-bosch.de/SABIORK/">SABIO-RK</a> — A database of reaction kinetics</li>
<li><a class="externallink" href="http://chem-faculty.ucsd.edu/kraut/dhfr.html" rel="nofollow" title="http://chem-faculty.ucsd.edu/kraut/dhfr.html">Joseph Kraut&#39;s Research Group, University of California San Diego</a> — Animations of several enzyme reaction mechanisms</li>
<li><a class="externallink" href="http://www.chem.qmul.ac.uk/iubmb/kinetics/" rel="nofollow" title="http://www.chem.qmul.ac.uk/iubmb/kinetics/">Symbolism and Terminology in Enzyme Kinetics</a> — A comprehensive explanation of concepts and terminology in enzyme kinetics</li>
<li><a class="externallink" href="http://orion1.paisley.ac.uk/kinetics/contents.html" rel="nofollow" title="http://orion1.paisley.ac.uk/kinetics/contents.html">An introduction to enzyme kinetics</a> — An accessible set of on-line tutorials on enzyme kinetics</li>
<li><a class="externallink" href="http://www.wiley.com/college/pratt/0471393878/student/animations/enzyme_kinetics/index.html" rel="nofollow" title="http://www.wiley.com/college/pratt/0471393878/student/animations/enzyme_kinetics/index.html">Enzyme kinetics animated tutorial</a> — An animated tutorial with audio</li></ul>
<p>{{featured article}}
{{Enzymes}}</p>
<p>
</p>
<p>{{Link FA}}
<a href="http://ca.wikipedia.org/wiki/Cin%C3%A8tica_enzim%C3%A0tica">ca:Cinètica enzimàtica</a>
<a href="http://de.wikipedia.org/wiki/Enzymkinetik">de:Enzymkinetik</a>
<a href="http://es.wikipedia.org/wiki/Cin%C3%A9tica_enzim%C3%A1tica">es:Cinética enzimática</a>
<a href="http://fr.wikipedia.org/wiki/Cin%C3%A9tique_enzymatique">fr:Cinétique enzymatique</a>
<a href="http://sr.wikipedia.org/wiki/Е%D0%BD%D0%B7%D0%B8%D0%BC%D1%81%D0%BA%D0%B0_%D0%BA%D0%B8%D0%BD%D0%B5%D1%82%D0%B8%D0%BA%D0%B0">sr:Ензимска кинетика</a>
<a href="http://zh.wikipedia.org/wiki/酶%E5%8B%95%E5%8A%9B%E5%AD%B8">zh:酶動力學</a>
</p>